<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="">
    <meta name="author" content="">
    <title>Home | Moreau Clara </title>
    <link href="css/bootstrap.min.css" rel="stylesheet">
    <link href="css/font-awesome.min.css" rel="stylesheet">
    <link href="css/animate.min.css" rel="stylesheet"> 
    <link href="css/lightbox.css" rel="stylesheet"> 
	<link href="css/main.css" rel="stylesheet">
	<link href="css/responsive.css" rel="stylesheet">  
	
    <link rel="shortcut icon" href="images/ico/favicon.ico">
	
    <link rel="stylesheet" href="css/academicons.min.css">

   </head><!--/head-->

<body>
	<header id="header">       
        <div class="container">
            <div class="row">
                <div class="col-sm-20 overflow">
                   <div class="social-icons pull-right">
                        <ul class="nav nav-pills">
                            <li><a href="https://twitter.com/ClaraMoreau9"><i class="fa fa-twitter"></i></a></li>
                            <li><a href="https://scholar.google.fr/citations?user=tkN9j_cAAAAJ&hl=fr&oi=ao"><i class="ai ai-google-scholar-square ai-1x"></i></a></li>
                            <li><a href="https://www.linkedin.com/in/clara-a-moreau-b44b6858/"><i class="fa fa-linkedin"></i></a></li>
                            <li><a href="https://www.facebook.com/clara.moreau.94"><i class="fa fa-facebook"></i></a></li>
                        </ul>
                    </div> 
                      <div class="slide-text">
                        <h1>Neuroimaging, Psychiatry and Genomics</h1>
                        <h2>Clara A. Moreau, Personal page</h2> 
                          <br>
                      </div>
                       <div class="slide-text">

                        <p><img src="images/clara.png" align=right>
				Polygenicity and genetic heterogeneity pose great challenges for studying mechanisms - such as brain connectivity - underlying neurodevelopmental psychiatric conditions. <br>
				Genetic-first approaches have been implemented in neuroimaging studies to address this issue of heterogeneity. How psychiatric genomic risk with various polygenicity levels impact brain features remains largely unknown. <br>
				Our work aims to understand the effect on the human brain of neurodevelopmental psychiatric disorders genomic risk factors with various levels of complexity: oligo-, multi-genic copy number variants (CNVs), and polygenic risk scores (PRS) as well as idiopathic psychiatric conditions and traits.<br>
				</p> 
                    
                           </div> 
                </div>
             </div>
        </div>
           </header>
    <!--/#header-->
    

    <!--/#home-slider-->
    
    
    <section id="services">
        <div class="container">
            <div class="row">
                <div id="tab-container">
                <h2 class="page-header">Curriculum vitae</h2>
                    <div class="col-md-6">
                        <ul id="tab" class="nav nav-tabs">
                            <li class="active"><a href="#tab1-item1" data-toggle="tab">Education <img src="images/academic.png"></a></li>
                            <li><a href="#tab1-item2" data-toggle="tab">Poster<img src="images/teaching.png"></a></li>
                            <li><a href="#tab1-item3" data-toggle="tab">Workshop and Teaching<img src="images/workshop.png"></a></li>


                        </ul>
                        <div class="tab-content">
                            <div class="tab-pane fade active in" id="tab1-item1">
                                <li>2021-: <b>Postdoctoral researcher</b>, <a href="https://research.pasteur.fr/fr/team/human-genetics-and-cognitive-functions/">Human genetics and cognitive functions</a>, Pasteur Institute <img src="images/FR.png"></li>
                                <p> Advisors: <a href="https://research.pasteur.fr/en/team/human-genetics-and-cognitive-functions/">Thomas Bourgeron</a> and <a href="https://www.institutmontaigne.org/experts/richard-delorme"> Richard Delorme</a> <br>
				<li>2020: <b>Postdoctoral researcher</b>, <a href="https://simexp.github.io/lab-website/team.html">Unité de neuroimagerie fonctionnelle</a>, CRIUGM <img src="images/ca.png"></li>
                                <p> Advisors: Pierre Bellec<br>
                                <li>2015-2020: <b>PhD in Neurosciences</b>,  University of Montreal <img src="images/ca.png"></li>
                                <p><a href="https://papyrus.bib.umontreal.ca/xmlui/handle/1866/24592">Mapping genome-wide neuropsychiatric mutation effects on functional brain connectivity</a>: Copy number variants delineate dimensions contributing to autism and schizophrenia.<br>
                                Co-supervised by: <a href="https://recherche.chusj.org/fr/Axes-de-recherche/Bio?id=97a89db2-d642-4ad8-a857-058b0a6d088d">Sébastien Jacquemont</a> (Geneticist, Sainte Justine Hospital) and <a href="https://simexp.github.io/lab-website/">Pierre L. Bellec</a> (Computer Science Department, SIMEXP lab, CR-IUGM, University of Montreal)</p>
                                <li>2019: <i>PhD Internship</i> (2 months) <a href="https://igc.ini.usc.edu/">Imaging Genetic Center</a> <img src="images/usa.png"></li>
                                <p>Advisors: P.M. Thompson, University of South California<br>
                                Skills: DTI analyses (ENIGMA) - Cross-CNVs data</p>
                                <li>2014-2015: <b>Research Assistant</b>, Medical Genetics Department, <a href="https://www.chuv.ch/fr/chuv-home/">CHUV</a>, <img src="images/ch.png"></li>
                                <p>Skills: MRI Protocol development and Scanning, Analyses (R, Matlab), Website development for family recruitment, neuropsychological assessment. S. Jacquemont’s laboratory</p>
                                <HR size=1 align=center width="80%">
                                <li>2012-2014: <b>Master degree in Cognitive Sciences</b> <a href="https://cogmaster.ens.psl.eu/en">Cogmaster </a><img src="images/FR.png"></li>
                                <p>Ecole Normale Supérieure and Descartes University, Paris</p>
                                <li>2013-2014 <i>Master Internship</i> Neurospin Institute, CEA <img src="images/FR.png"></li>
                                <p>Advisors: Marion Noulhiane and Lucie Hertz-Pannier <a href="http://joliot.cea.fr/drf/joliot/Pages/Entites_de_recherche/neurospin/UNIACT/Equipe-InDEv.aspx">UNIACT</a><br>
                                Skills: fMRI analysis (SPM, Matlab), Neurodevelopmental cohort, memory tasks.</p>
                                <li>2013 <i>Summer Internship</i> University of Montreal <img src="images/ca.png"></li> 
                                <p>Advisor: Pr. P. Jolicoeur, Neuroscience department<br>
                                Skills: MEG/EEG analyses for an auditory task.</p>
                                <li>2012-2013 <i>Master Internship</i> INSERM – UMR 663 Paris, France</li>
                                <p>Advisors: Pr. M. Noulhiane and Dr C. Chiron, Necker Children Hospital<br>
                                <HR size=1 align=center width="80%">
                                <li>2009-2012: <b>BSc degree in Neuropsychology</b> <img src="images/FR.png"></li>
                                <p>Descartes University, Paris-V </p>
                                <li>2012 <i> Summer Internship</i> <a href="https://crnl.univ-lyon1.fr/index.php/fr/Recherche/Equipes/4">Codage et Mémoire Olfactive</a> Lyon, France </li>
                                <p>Advisor: Pr. N. Ravel and Pr. Rémi Gervais<br> 
                                Skills: Recording olfactory cells in mice (electrophysiology) and signal processing</p>        
                                <HR size=1 align=center width="80%">

                                <li>2006-2009: <b>Scientific Baccalaureate</b> <img src="images/FR.png"></li>
                                <p>Victor Duruy High School, Paris 75007 France</p>  <br>
                            </div>

                            <div class="tab-pane fade" id="tab1-item2">
                                <li>C. Moreau, K. Kumar, A. Harvey, …P. Bellec, S. Jacquemont. <br> <a href='https://www.anyscreeninc.com/PF/OHBM/2021/OHBM-Educational-Courses/pdf_poster_files/Clara_Moreau60785c7f6b8aa/Clara_Moreau.pdf'> Atlas of functional connectivity relationships across rare and common genetic variants, traits, and psychiatric conditions</a><br>OHBM 2021 </li>
				<li>C. Moreau, S. Urchs, G. Huguet, …P. Bellec, S. Jacquemont. <br>Functional Connectivity Analyses Suggest Shared Molecular Mechanisms Across 12 Neuropsychiatric Mutations, Autism and Schizophrenia. <br>SOBP 2020 <img src="images/usa.png"></li>
                                <li>C. Moreau, S. Urchs, G. Huguet, …P. Bellec, S. Jacquemont. <br>Brain-Wide Connectome Analyses Suggest Shared Mechanisms Across Eight High-Risk Neuropsychiatric Mutations. <br>INS, Denver, Colorado 2020 <img src="images/usa.png"></li>
                                <li>C. Moreau*, S. Urchs*, C. Schramm, …, Simons VIP Consortium, C. Bearden, P. Bellec, S. Jacquemont. <br>Gene dosage alters brain connectivity and delineates functional signatures contributing to autism and schizophrenia. <br>World Congress of Psychiatric Genetics Anaheim, California, USA 2019 <img src="images/usa.png"></li>
                                <li>C. Moreau*, C. Schramm*, G. Huguet, S. Urchs, Kumar K., Douard E., A. Evans, Labbe A., Greenwood C., Chakravarty M., P. Bellec, S. Jacquemont. <br>Estimating the commonalities of any recurrent CNVs on different anatomical and functional brain metrics. <br>Human Brain Mapping, Rome 2019 <img src="images/it.png"></li>
                                <li>C. Bearden, D. Sun, A. Lin, C. Ching, S. Jacquemont, C. Moreau, J. Villalon. <br>Gene Dosage Effects on Neurobehavioral Phenotypes and Development: Relevance to Idiopathic Neuropsychiatric Disorders. <br>SOBP, Chicago 2019 <img src="images/usa.png"></li>
                                <li>C. Moreau*, S. Urchs*, C. Schramm, P.O. Quirion, A. Lin, L. Kushan, A. Evans, J.D. Lewis, Simons VIP Consortium, C. Bearden, P. Bellec, S. Jacquemont. <br>Shared functional connectivity alterations across neurodevelopmental mutations, ASD, ADHD and schizophrenia. <br>Enhancing Psychiatric Genetic Testing, and Training in Europe, EnGagE, Paris 2019 <img src="images/FR.png"></li> 
                                <li>C. Moreau*, S. Urchs*, C. Schramm, P.O. Quirion, A. Lin, L. Kushan, A. Evans, J.D. Lewis, Simons VIP Consortium, C. Bearden, P. Bellec, S. Jacquemont. <br>Mirror effects of 4 neurodevelopmental CNVs on functional connectivity and implication for idiopathic autism. <br>Sixth Biennial Conference on Brain Connectivity, Montreal 2018 <img src="images/ca.png"></li>
                                <li>A Jonch, E. Douard, C. Moreau, … S. Jacquemont. <br>The true contribution of the 15q11.2 BP1-BP2 deletion to neurodevelopmental symptoms. <br>World Congress of Psychiatric Genetics, Glasgow 2018 <img src="images/scot.png"></li>
                                <li>C. Moreau*, S. Urchs*, … , A. Evans, J. D. Lewis, P. Bellec, S. Jacquemont. <br>Mirror effects of 4 neurodevelopmental CNVs on functional connectivity and implication for idiopathic autism  <br>World Congress of Psychiatric Genetics, Glasgow 2018 <img src="images/scot.png"></li>
                                <li>C. Moreau*, S. Urchs*, … , A. Evans, J. D. Lewis, P. Bellec, S. Jacquemont. <br>Global functional over-connectivity in 16p11.2 CNV deletion carriers. <br>Human Brain Mapping, Singapore 2018 <img src="images/singap.png"></li>
                                <li>C. R. Pernet, D Rodriguez, C. Moreau, D. Marinazzo, A. Eklund. <br>European Network for Brain Imaging of Tumours. <br>Neuroinformatics, Montreal, 2018 <img src="images/ca.png"></li>
                                <li>C. Moreau*, S. Urchs*, Simons Variation in Individuals Project Consortium, A. Evans, J. D. Lewis, P. Bellec, S. Jacquemont. <br>Altered brain connectivity in patient with 16p11.2. <br>World Congress of Psychiatric Genetics, Orlando 2017 <img src="images/usa.png"></li>
                                <li>C. Moreau*, J.D. Lewis*, A. Evans‡, S. Jacquemont‡, and the Simons Variation in Individuals Project Consortium. <br>Altered subcortical diffusivity in 16p11.2 CNVs. <br>Human Brain Mapping, Vancouver 2017 <img src="images/ca.png"></li>
                                <li>J.D. Lewis*, C. Moreau*, S. Martin-Brevet, …, S. Jacquemont‡, A. Evans‡, the 16p11.2 European Consortium, and the Simons VIP Consortium. <br>Thickness and contrast in 16p11.2 CNVs.  <br>Human Brain Mapping, Vancouver 2017 <img src="images/ca.png"></li>
                                <li>F. Chouinard-Decorte, P. Rioux, J. Lewis, C. Moreau, …, P. Bellec, D. Glahn, A. Evans. <br>Genetic clustering of the human functional connectome. <br>Human Brain Mapping, Vancouver 2017 <img src="images/ca.png"></li>
                                <li>S. Martin, B. Rodriguez-Herreros, J. Nielsen, C. Moreau, …, B. Draganski, S. Jacquemont. <br>The Effects of 16p11.2 Gene Dosage on Brain Structure. <br>IMFAR, San Francisco 2017 <img src="images/usa.png"></li>
                                <li>A. E. Jønch, I. Roberts-Caldeira, C. Moreau, …, S. Jacquemont. <br>Distal and proximal copy number variations at the 16p11.2 locus present similar anthropometric and phenotypic traits. <br>17th International Fragile X and other Early-Onset Cognitive Disorders Strasbourg France 2015 <img src="images/FR.png"></li>  

                            </div>
                            <div class="tab-pane fade" id="tab1-item3">
                                <h3>Workshop</h3>
				<li>08.2017: <b>Summer school Deep Learning</b> University of Montreal (MILA), CA</li>
                                <li>06.2017: <b>Workshop Brainhack</b> University of British Columbia, CA</li>
                                <li>08.2016: <b>Summer school Brain Imaging Genetics for Imagers</b> Radboud University, Nijmegen</li>
                                <li>06.2016: <b>Workshop Brainhack (fMRI)</b> - Lausanne, CH</li>
                                <li>02.2016: <b>Workshop Brainhack (MRI)</b> - Pasteur Institute, FR</li>
                                <li>04.2015: <b>HackTheBrain workshop (EEG)</b> - London, UK</li>
                                <li>08.2013: <b>Summer school Basic and Advanced functional MRI</b> McGill University, Montreal, CA</li>
                                <HR size=1 align=center width="80%">

                                <h3>Teaching - Students supervision</h3>
                               	<li>2022 (6 months): Main supervisor of Pierre Bergeret (Master in Bioinformatics, University of Paris Saclay)</li>
                               	<li>2022 (6 months): Main supervisor of Lydie Tran (Master in Bioinformatics, University of Paris Saclay)</li>
				<li>2022 (6 months): Co-supervisor of Alexis Debril (Master in Neurosciences, PSL)</li>					
				<li>12.2021: Course 'Neuroimagerie dans les troubles du spectre autistique' at the Robert Debré hospital</li>
				<li>2021 (6 months): Main supervisor of Louise Dry (AgroParisTech, Paris)</li>
				<li>2021 (4 months): Main supervisor of Swan Portalier (Magister de Génétique, Université de Paris Cité)</li>
			   	<li>2020-2022 (2 years): Co-supervisor of Annabelle Harvey (Master student, Informatics, UdeM)</li>
				<li>2019-2022 (3 years): Co-supervisor of Andreanne Proulx (Bachelor and Master degrees, in Psychology, UdeM)</li>
                                <li>2018-2020: Teaching Assistant (3 years), 
                                    <a href="https://psy.umontreal.ca/fileadmin/Documents/FAS/Psychologie/Documents/1-Programmes-cours/Plans_de_cours/Automne_2018/PSY3018B.pdf">Brain Imaging Techniques </a>(100 students, UdeM,  Pr. P. Bellec)</li>
                                <li>2019 (6 months): Main supervisor of Genevieve Dumais (BSc–3rd year, Neurosciences, UdeM)</li>
                                <li>06.2018: Instructor at the <a href="https://brainhackmtl.github.io/school2019/">Brainhack School 2018 (Imaging genetics, UdeM)</a></li>
                                <li>2016 (3 months): Main supervisor of Agathe Casgrain-Cyr (BSc–3rd year, Bioinformatics, UdeM)</li>
                                <HR size=1 align=center width="80%">    
                                <h3>Reviewing</h3>
				<li>2022: Thesis committee of Dominika Slušná (University of Pompeu-Fabra, Barcelona)</li>	
				<li>2021: Agence National de la recherche (ANR, France)(n=1)</li>	
                                <li>2020: Brain (n=1)</li>
				<li>2020: Molecular Autism (n=1)</li>
                                <li>2019: Neuroimage (n=3 papers)</li>
				<li>2020: Progress in Neuropsychopharmacology & Biological Psychiatry (n=1)</li>

					
				<h3>Editor</h3>
				<li>2021: Co-editor, Frontiers in Psychiatry</li>	
				<a href="https://www.frontiersin.org/research-topics/28305/combining-multimodal-brain-imaging-data-for-an-integrated-characterization-of-neurodevelopmental-con">Combining Multimodal Brain Imaging Data for an integrated characterization of Neurodevelopmental Conditions</a></li>                               
                            </div>
                            
                        </div>
                    </div>
                     <div class="col-md-6">
                        <ul id="tab" class="nav nav-tabs">
                            <li class="active"><a href="#tab2-item1" data-toggle="tab">Conferences <img src="images/conf.png"></a></li>
                            <li><a href="#tab2-item2" data-toggle="tab">Science Dissemination <img src="images/transfer.png"></a></li>
                            <li><a href="#tab2-item3" data-toggle="tab">Other infos <img src="images/sail.png"></a></li>
                        </ul>
                    
                        <div class="tab-content">
                            <div class="tab-pane fade active in" id="tab2-item1">
				<li>06.2022: OHBM, Glasgow</li>
				<p>Oral session <a href="https://www.humanbrainmapping.org/files/2022/Oral%20Sessions/Imaging_Genetics_Mapping_the_Effects_of_Genetic_and_Transcriptional_Variation_on_the_Brain.pdf">“Imaging Genetics: Mapping the Effects of Genetic and Transcriptional Variation on the Brain”</a></p>				
				<li>05.2022: Seminar, McGill University, CA</li>
				<p> “Genetic heterogeneity and pleiotropy shape brain connectivity in psychiatric conditions”</a></p>
				<li>12.2021: Académie de Medecine</li>
				<p>Seminaire <a href="https://itneuro.aviesan.fr/openscience2021.html">“Troubles du neurodéveloppement sans frontière”</a></p>
				<li>12.2021: IDA</li>  
                                <p>Seminaire “Genetic heterogeneity in neurodevelopmental conditions shapes brain connectivity”</a></p>
                                <li>10.2021: European Congress Neuropsychopharmacology, Lisboa</li>
                                <p>Symposium <a href="https://www.ecnp.eu/Congress2021/ECNPcongress/programme/Programme#!sessiondetails/0000092200_0">“The genetics of autism from risk to resilience”</a></p>
				 <li>05.2021: Les enjeux actuels en Neuroéducation</li>
                                <p>Seminaire <a href="https://www.associationneuroeducation.org/serie-de-conferences-2021">“Que nous ont appris les dernières avancées en neuro-imagerie et génétique sur les troubles du spectre de l'autisme?”</a></p>
				<li>04.2021: Society of Biological Psychiatry, virtual meeting</li>
                                <p>Symposium <a href="https://sobp.societyconference.com/conf/#sessions/conf10007">“Brain Alterations and Mechanisms in Carriers of Genomic Structural Variants”</a></p>   
                            	<li>02.2021: Pasteur Institute</li>
                                <p>Lecture “Atlas of functional connectivity relationships across rare and common genetic variants, traits, and psychiatric conditions”</p>       
				<li>06.2020: Human Brain Mapping, virtual meeting</li>
                                <p><a href="https://www.humanbrainmapping.org/files/2020/OHBM_Neuropsychiatric.pdf">Symposium “Neuropsychiatric genetic variation shapes brain architecture by modulating gene expression”</a></p>   
                                <li>03.2020: Feindel BIC Lecture McGill University, CA</li>
                                <p><a href="https://www.mcgill.ca/neuro/channels/event/feindel-virtual-brain-and-mind-seminar-neuro-mapping-neuropsychiatric-mutation-effects-functional-321797">Mapping neuropsychiatric mutation effects on functional brain connectivity to delineate dimensions contributing to autism and schizophrenia</a></p>   
                                <li>08.2019: Imaging Genetic Center Lecture, University South California, USA</li>
                                <p><a href="https://www.facebook.com/photo.php?fbid=10156383123046892&set=pb.664156891.-2207520000..&type=3">High-risk psychiatric mutations modulate functional brain connectivity pointing to dimensions involved in autism and schizophrenia</a></p>
                                <li>06.2019: Human Brain Mapping, Rome, IT</li>
                                <p>Symposium <a href="https://www.pathlms.com/ohbm/courses/12238/sections/15843/video_presentations/138313">A tough nut to crack: neurodevelopmental connectopathies</a></p>  
                                <li>05.2019: Society of Biological Psychiatry, Chicago, USA</li>
                                <p>Symposium “Large Scale Imaging Studies of Rare Copy Number variants: Brain Imaging from Enigma and Other Large-Scale International Studies”</p>
                                <li>04.2019: International Society for Autism Research, Montreal, CA</li>
                                <p>Symposium “Human and Animal Models: Impact of High-Risk Copy Number Variants on Brain Structure, Functional Connectivity, and Sexual Development.”</p>        
                                <li>05.2016: University of Montreal, CA</li>
                                <p>Lecture "From the first human genome to recent application in genome edition: ethical issues.”(45 min)</p>  
                                <li>11.2014: EPFL, CH</li>
                                <p>Symposium <a href="https://www.epfl-innovationpark.ch/epfled2015/protecting-ideas-liberating-innovation/">Protecting Ideas, Liberating Innovation, and Open collaboration</a></p>              
                                <li>09.2014: University of Lausanne, CH</li>
                                <p>Symposium <a href="https://www.youtube.com/watch?v=fs4-xMb-4Kc&feature=youtu.be">Open Access, Open Data and Open Science: reality, and amalgam</a></p> 
                                <li>03.2014: Annual Cognitive science forum, Paris, FR</li>
                                <p>Symposium <a href="https://www.youtube.com/watch?v=CHJVMPAQcXo">Artificial, animal, and human cognitive processes: How could we delineate Intelligence?</a></p> 
                                <li>03.2014: Cité des Sciences et de l’Industrie, Paris, FR</li>
                                <p>Symposium Conciliate Research, Innovation and Open Science</p> 
                            </div>
                            
                            <div class="tab-pane fade" id="tab2-item2">
                                <li>05.2021: <a href="https://www.spectrumnews.org/opinion/qa-with-sebastien-jacquemont-and-clara-moreau-why-brain-imaging-signatures-for-autism-are-so-elusive/">Simons Foundation Autism Research Initiative</a></li> 
				<p> Q&A with Sébastien Jacquemont and Clara Moreau: Why brain imaging signatures for autism are so elusive</p>
				<li>11.2020: <a href="https://www.spectrumnews.org/news/gene-mutations-point-to-overlaps-in-brain-connectivity-for-autism-schizophrenia/">Simons Foundation Autism Research Initiative</a></li> 
				<p> Gene mutations point to overlaps in brain connectivity for autism, schizophrenia</p>
				<li>10.2020 - <a href="https://recherche.chusj.org/fr/Communications/Nouvelles/2020/Genes-connectivite-cerebrale-et-maladies-neurops">Communiqué de presse - CHU Ste Justine</a></li> 
				<p>Gènes, connectivité cérébrale, et maladies neuropsychiatriques </p>
				<li>2019: BIDS contributor <a href="https://bids-specification.readthedocs.io/en/latest/06-extensions.html"> Brain Imaging Data Structure</a></li> 
                                <p>Genomic information (C. Pernet).</p>
                                <li>06.2016: <a href="http://www.brainhack.org/OHBM2016/">Brain imaging workshop</a> (Brainhack) </li> 
                                <p> Co-organizer with Pierre Bellec, Lausanne, Switzerland</p>
                                <li>2016: Part of the science dissemination team</li>
                                <p><a href="https://recherche.chusj.org/fr/Blogue/Clara-Moreau">Brain and Development research</a> section at the Ste Justine Hospital, Montreal</p>
                                <li>2015-2016: Radio show Podcast Science</li>
                                <p><a href="http://www.podcastscience.fm/emission/2015/01/20/podcast-science-202-les-maladies-mentales">Mental Ilnesses</a> <br> <a href="https://www.podcastscience.fm/emission/2013/11/24/podcast-science-152-dans-le-cerveau/">Inside the Brain (Neuroimaging techniques)</a></p>
                                <li>2014-2015: <a href="http://republic-of-innovation.ch/startinnov-de-lidee-a-la-creation-dentreprise-un-pas-a-franchir-durant-une-journee-de-simulation/">Startup Weekend</a> co-organizer</li>
                                <p>For PhD students, n=6 editions <br> 
                                 Lausanne (EPFL), and Paris (ENS, CogInnov, and ESPCI). </p>
                                <li>2014-2015: Board member and web manager for <a href="http://sciencesenmarche.org/fr/paris/qui-sommes-nous/">Science en marche</a></li>
                                <p>National political committee <br> French organization of researchers</p>
                                <li>2014: Conference co-organizer at the ENS University of Paris</li>
                                <p>“Conciliate Open Science, Patents, and Intellectual property issues”</p>
                                <li>2014-2015: Board member of <a href="http://lapaillasse.org/presentation">La Paillasse</a></li>
				 <p>Citizen lab in Paris</p>
                                <li>06.2015: Open innovation <a href="https://www.liftglobal.org/about">Lift Conference</a>, Shanghai, China</li>
                                <li>03.2013: Co-organizer of the annual french <a href="https://www.youtube.com/watch?v=CHJVMPAQcXo">Cognitive sciences Forum</a></li>
                                <p>Consciousness process of intelligence across the animal, the human, and the computer</p>
                                <li>2013-2015: Secretary of <a href="http://hackyourphd.org/qui-sommes-nous/">Hack your PhD</a> association</li>
                                <p>To promote Open Access of scientific publications</p>
                                <p>Open Knowledge Conference, Geneva, Switzerland</p>
                                <li>2012-2015: Active member of “Cognivence” and <a href="https://fresco.scicog.fr/valorisation">FRESCO</a> » </li>
                                <p>French Federation of Cognitive Science students)</p>

                            </div>
                            <div class="tab-pane fade" id="tab2-item3">
                                <h3>Computer skills</h3>
                                <li>Programming  languages: R, Python, bash</li>
                                <li>Softwares: NIAK (fMRI), fMRIprep, SPM12 (VBM, Neuromorphometric), Freesurfer and Civet (Cortical Thickness),  FSL (TBSS)<br>
                                <a href= "https://github.com/claramoreau9">Github</a>, Jupyter Notebook</li>
                                <HR size=1 align=center width="80%">
                                <h3>Language</h3>
                                    <li>French (native)</li>
                                    <li>English (fluent)</li>
                                <HR size=1 align=center width="80%">
                                <h3>Other skills</h3>
                                    <li>Driving license: Car (2011), Motorboat (2016), Sailboat (2017)</li>
                                    <li>Diving license (PADI, 2018).</li>
                                <h3>Jobs</h3>
                                <li>Summers 2010, 2011 and 2012: Saleswoman in Paris, “Berthillon” (full-time)</li>
                                <li>Sept 2010 to June 2012: Waitress in a restaurant in Paris (half-time)</li>  
                                <h3>Hobbies</h3>
                                    <li>Hiking, Sailing, Travelling, Drinking french wine</li>
					<br><br>
                                    <p align= center><img  src="images/voile.png"></p>
                             </div>
                        </div>
                    </div>
                </div>
            </div>

    <!--/#page-breadcrumb-->

    <section id="services">
        <div class="container">
            <div class="row">
                <div id="accordion-container">
                <h2 class="page-header"><img src="images/article.png">   Articles  </h2> 
                 
                <div class="panel-group" id="accordion">
                    <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
                    
		<div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapse7">
                                    Quantifying the Effects of 16p11.2 Copy Number Variants on Brain Structure: A Multisite Genetic-First Study.
                                </a>
                                <h5> <a target="_blank" href="https://doi.org/10.1016/j.biopsych.2018.02.1176">Biological Psychiatry 2018</a></h5>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.1016/j.biopsych.2018.02.1176"></div>

                            </h4>
                        </div>
                       <div id="collapse7" class="panel-collapse collapse">
                            <div class="panel-body">
                                <p>Martin-Brevet, S., Rodríguez-Herreros, B., Nielsen, J. A., Moreau, C., Modenato, C., Maillard, A. M., Pain, A., Richetin, S., Jønch, A. E., Qureshi, A. Y., Zürcher, N. R., Conus, P., 16p11.2 European Consortium, Simons Variation in Individuals Project (VIP) Consortium, Chung, W. K., Sherr, E. H., Spiro, J. E., Kherif, F., Beckmann, J. S., … Jacquemont, S. </p>
                                
                                 <br>
                                <h4>Abstract</h4>

                                <p>16p11.2 breakpoint 4 to 5 copy number variants (CNVs) increase the risk for developing autism spectrum disorder, schizophrenia, and language and cognitive impairment. In this multisite study, we aimed to quantify the effect of 16p11.2 CNVs on brain structure. <br>Using voxel- and surface-based brain morphometric methods, we analyzed structural magnetic resonance imaging collected at seven sites from 78 individuals with a deletion, 71 individuals with a duplication, and 212 individuals without a CNV. <br>Beyond the 16p11.2-related mirror effect on global brain morphometry, we observe regional mirror differences in the insula (deletion > control > duplication). Other regions are preferentially affected by either the deletion or the duplication: the calcarine cortex and transverse temporal gyrus (deletion > control; Cohen’s d > 1), the superior and middle temporal gyri (deletion  > control; Cohen’s d inf −1), and the caudate and hippocampus (control > duplication; −0.5 > Cohen’s d > −1). Measures of cognition, language, and social responsiveness and the presence of psychiatric diagnoses do not influence these results. The global and regional effects on brain morphometry due to 16p11.2 CNVs generalize across site, computational method, age, and sex. Effect sizes on neuroimaging and cognitive traits are comparable. Findings partially overlap with results of meta-analyses performed across psychiatric disorders. However, the lack of correlation between morphometric and clinical measures suggests that CNV-associated brain changes contribute to clinical manifestations but require additional factors for the development of the disorder. <br>These findings highlight the power of genetic risk factors as a complement to studying groups defined by behavioral criteria.</p>
                            </div>
                        </div>
                    </div>  
			
		<div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapse8">
                                    Dose response of the 16p11.2 distal copy number variant on intracranial volume and basal ganglia.
                                </a>
                                <h5> <a target="_blank" href="https://doi.org/10.1038/s41380-018-0118-1">Molecular Psychiatry 2018</a></h5>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.1038/s41380-018-0118-1"></div>

                            </h4>
                        </div>         
                       <div id="collapse8" class="panel-collapse collapse">
                            <div class="panel-body">
                                <p>Sønderby, I. E., Gústafsson, Ó., Doan, N. T., Hibar, D. P., Martin-Brevet, S., Abdellaoui, A., Ames, D., Amunts, K., Andersson, M., Armstrong, N. J., Bernard, M., Blackburn, N., Blangero, J., Boomsma, D. I., Bralten, J., Brattbak, H.-R., Brodaty, H., Brouwer, R. M., Bülow, R., … Moreau, C., … 16p11.2 European Consortium, for the ENIGMA-CNV working group.</p>
                                
                                 <br>
                                <h4>Abstract</h4>

                                <p>Carriers of large recurrent copy number variants (CNVs) have a higher risk of developing neurodevelopmental disorders. The 16p11.2 distal CNV predisposes carriers to e.g., autism spectrum disorder and schizophrenia. <br>We compared subcortical brain volumes of 12 16p11.2 distal deletion and 12 duplication carriers to 6882 non-carriers from the large-scale brain Magnetic Resonance Imaging collaboration, ENIGMA-CNV. <br>After stringent CNV calling procedures, and standardized FreeSurfer image analysis, we found negative dose-response associations with copy number on intracranial volume and on regional caudate, pallidum and putamen volumes (β = -0.71 to -1.37; P  inf 0.0005). In an independent sample, consistent results were obtained, with significant effects in the pallidum (β = -0.95, P = 0.0042). The two data sets combined showed significant negative dose-response for the accumbens, caudate, pallidum, putamen and ICV (P = 0.0032, 8.9 × 10-6, 1.7 × 10-9, 3.5 × 10-12 and 1.0 × 10-4, respectively). Full scale IQ was lower in both deletion and duplication carriers compared to non-carriers. <br>This is the first brain MRI study of the impact of the 16p11.2 distal CNV, and we demonstrate a specific effect on subcortical brain structures, suggesting a neuropathological pattern underlying the neurodevelopmental syndromes.</p>
                            </div>
                        </div>
                      </div> 
			
			  <div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapse10">
                                    Association of Copy Number Variation of the 15q11.2 BP1-BP2 Region With Cortical and Subcortical Morphology and Cognition.
                                </a>
                                <h5> <a target="_blank" href="https://doi.org/10.1001/jamapsychiatry.2019.3779">JAMA Psychiatry 2019</a></h5>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.1001/jamapsychiatry.2019.3779"></div>

                            </h4>
                        </div>
                        <div id="collapse10" class="panel-collapse collapse">
                            <div class="panel-body">
                                <p>van der Meer, D., Sønderby, I. E., Kaufmann, T., Walters, G. B., Abdellaoui, A., Ames, D., Amunts, K., Andersson, M., Armstrong, N. J., Bernard, M., Blackburn, N. B., Blangero, J., Boomsma, D. I., Brodaty, H., Brouwer, R. M., Bülow, R., Cahn, W., Calhoun, V. D., Caspers, S., ..., Moreau C., … Andreassen, O. A.</p>
                                 <br>
                                <h4>Abstract</h4>

                                <p>Recurrent microdeletions and duplications in the genomic region 15q11.2 between breakpoints 1 (BP1) and 2 (BP2) are associated with neurodevelopmental disorders. These structural variants are present in 0.5% to 1.0% of the population, making 15q11.2 BP1-BP2 the site of the most prevalent known pathogenic copy number variation (CNV). It is unknown to what extent this CNV influences brain structure and affects cognitive abilities. To determine the association of the 15q11.2 BP1-BP2 deletion and duplication CNVs with cortical and subcortical brain morphology and cognitive task performance. <br>In this genetic association study, T1-weighted brain magnetic resonance imaging were combined with genetic data from the ENIGMA-CNV consortium and the UK Biobank, with a replication cohort from Iceland. In total, 203 deletion carriers, 45 247 noncarriers, and 306 duplication carriers were included. Data were collected from August 2015 to April 2019, and data were analyzed from September 2018 to September 2019. <br>The associations of the CNV with global and regional measures of surface area and cortical thickness as well as subcortical volumes were investigated, correcting for age, age2, sex, scanner, and intracranial volume. Additionally, measures of cognitive ability were analyzed in the full UK Biobank cohort. Of 45 756 included individuals, the mean (SD) age was 55.8 (18.3) years, and 23 754 (51.9%) were female. Compared with noncarriers, deletion carriers had a lower surface area (Cohen d = -0.41; SE, 0.08; P = 4.9 × 10-8), thicker cortex (Cohen d = 0.36; SE, 0.07; P = 1.3 × 10-7), and a smaller nucleus accumbens (Cohen d = -0.27; SE, 0.07; P = 7.3 × 10-5). There was also a significant negative dose response on cortical thickness (β = -0.24; SE, 0.05; P = 6.8 × 10-7). Regional cortical analyses showed a localization of the effects to the frontal, cingulate, and parietal lobes. Further, cognitive ability was lower for deletion carriers compared with noncarriers on 5 of 7 tasks. <br>These findings, from the largest CNV neuroimaging study to date, provide evidence that 15q11.2 BP1-BP2 structural variation is associated with brain morphology and cognition, with deletion carriers being particularly affected. The pattern of results fits with known molecular functions of genes in the 15q11.2 BP1-BP2 region and suggests involvement of these genes in neuronal plasticity. These neurobiological effects likely contribute to the association of this CNV with neurodevelopmental disorders.</p>
                            </div>
                        </div>    
                      </div>
                    
                      <div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapse11">
                                    Developmental trajectories of neuroanatomical alterations associated with the 16p11.2 Copy Number Variations.
                                </a>
                                <h5> <a target="_blank" href="https://doi.org/10.1016/j.neuroimage.2019.116155">NeuroImage 2019</a></h5>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.1016/j.neuroimage.2019.116155"></div>

                            </h4>
                        </div>
                        <div id="collapse11" class="panel-collapse collapse">
                            <div class="panel-body">
                                <p>Cárdenas-de-la-Parra, A., Martin-Brevet, S., Moreau, C., Rodriguez-Herreros, B., Fonov, V. S., Maillard, A. M., Zürcher, N. R., 16p11.2 European Consortium, Hadjikhani, N., Beckmann, J. S., Reymond, A., Draganski, B., Jacquemont, S., and Collins, D. L. </p>
                                
                                 <br>
                                <h4>Abstract</h4>

                                <p>Most of human genome is present in two copies (maternal and paternal). However, segments of the genome can be deleted or duplicated, and many of these genomic variations (known as Copy Number Variants) are associated with psychiatric disorders. 16p11.2 copy number variants (breakpoint 4-5) confer high risk for neurodevelopmental disorders and are associated with structural brain alterations of large effect-size. Methods used in previous studies were unable to investigate the onset of these alterations and whether they evolve with age. <br>In this study, we aim at characterizing age-related effects of 16p11.2 copy number variants by analyzing a group with a broad age range including younger individuals. A large normative developmental dataset was used to accurately adjust for effects of age. We normalized volumes of segmented brain regions as well as volumes of each voxel defined by tensor-based morphometry. <br>Results show that the total intracranial volumes, the global gray and white matter volumes are respectively higher and lower in deletion and duplication carriers compared to control subjects at 4.5 years of age. These differences remain stable through childhood, adolescence and adulthood until 23 years of age (range: 0.5 to 1.0 Z-score). Voxel-based results are consistent with previous findings in 16p11.2 copy number variant carriers, including increased volume in the calcarine cortex and insula in deletions, compared to controls, with an inverse effect in duplication carriers (1.0 Z-score). All large effect-size voxel-based differences are present at 4.5 years and seem to remain stable until the age of 23. <br>Our results highlight the stability of a neuroimaging endophenotype over 2 decades during which neurodevelopmental symptoms evolve at a rapid pace.</p>
                            </div>
                        </div>
                      </div> 
                                                     
                    <div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapse12">
                                    Estimating the effect size of the 15q11.2 BP1-BP2 deletion and its contribution to neurodevelopmental symptoms: recommendations for practice.
                                </a>
                                <h5> <a target="_blank" href="https://doi.org/10.1136/jmedgenet-2018-105879">Journal of Medical Genetics 2019</a></h5>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.1136/jmedgenet-2018-105879"></div>

                            </h4>
                        </div>
                        <div id="collapse12" class="panel-collapse collapse">
                            <div class="panel-body">
                                <p>Jønch, A. E., Douard, E., Moreau, C., Van Dijck, A., Passeggeri, M., Kooy, F., Puechberty, J., Campbell, C., Sanlaville, D., Lefroy, H., Richetin, S., Pain, A., Geneviève, D., Kini, U., Le Caignec, C., Lespinasse, J., Skytte, A.-B., Isidor, B., Zweier, C., ..., Ousager LB.,  Jacquemont, S., on behalf of 15q11.2 Working Group </p>
                                
                                 <br>
                                <h4>Abstract</h4>

                                <p>The 15q11.2 deletion is frequently identified in the neurodevelopmental clinic. Case–control studies have associated the 15q11.2 deletion with neurodevelopmental disorders, and clinical case series have attempted to delineate a microdeletion syndrome with considerable phenotypic variability. The literature on this deletion is extensive and confusing, which is a challenge for genetic counselling. The aim of this study was to estimate the effect size of the 15q11.2 deletion and quantify its contribution to neurodevelopmental disorders. <br>We performed meta-analyses on new and previously published case–control studies and used statistical models trained in unselected populations with cognitive assessments. We used new (n=241) and previously published (n=150) data from a clinically referred group of deletion carriers. <br>15q11.2 duplications (new n=179 and previously published n=35) were used as a neutral control variant. The deletion decreases IQ by 4.3 points. The estimated ORs and respective frequencies in deletion carriers for intellectual disabilities, schizophrenia and epilepsy are 1.7 (3.4%), 1.5 (2%) and 3.1 (2.1%), respectively. There is no increased risk for heart malformations and autism. In the clinically referred group, the frequency and nature of symptoms in deletions are not different from those observed in carriers of the 15q11.2 duplication suggesting that most of the reported symptoms are due to ascertainment bias. <br>We recommend that the deletion should be classified as ‘pathogenic of mild effect size’. Since it explains only a small proportion of the phenotypic variance in carriers, it is not worth discussing in the developmental clinic or in a prenatal setting.</p>
                            </div>
                        </div>
                      </div> 
			
			  <div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapse9">
                                    MIST: A multi-resolution parcellation of functional brain networks.
                                </a>
                                <h5> <a target="_blank" href="https://doi.org/10.12688/mniopenres.12767.2">MNI Open Research 2019</a></h5>

                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.12688/mniopenres.12767.1"></div>

                            </h4>
                        </div>
                        <div id="collapse9" class="panel-collapse collapse">
                            <div class="panel-body">
                                <p>Urchs, S., Armoza, J., Moreau, C., Benhajali, Y., St-Aubin, J., Orban, P., Bellec, P. </p>
                                <br>
                                <h4>Abstract</h4>

                                <p>The functional architecture of the brain is organized across multiple levelsof spatial resolutions, from distributed networks to the localized areas theyare made of. A brain parcellation that defines functional nodes at multipleresolutions is required to investigate the functional connectome acrossthese scales. <br> Here we present the Multiresolution Intrinsic SegmentationTemplate (MIST), a multi-resolution group level parcellation of the cortical,subcortical and cerebellar gray matter. The individual MIST parcellationsmatch other published group parcellations in internal homogeneity andreproducibility and perform very well in real-world application benchmarks.In addition, the MIST parcellations are fully annotated and provide ahierarchical decomposition of functional brain networks across nineresolutions (7 to 444 functional parcels). We hope that the MISTparcellation will accelerate research in brain connectivity acrossresolutions. Because visualizing multiresolution parcellations is challenging,we provide  to explore the MIST. The MIST isan interactive web interfacealso available through the popular toolbox.</p>
                            </div>
                        </div>    
                      </div>  
			
	              <div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapse4">
                                    Differential effects of deletions and duplications on autism risk across the genome. 
                                </a>
                                
                                <h5> <a target="_blank" href="https://ajp.psychiatryonline.org/doi/10.1176/appi.ajp.2020.19080834">American Journal of Psychiatry 2020</a> </h5>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.1176/appi.ajp.2020.19080834"></div>

                            </h4>
                        </div>
                        <div id="collapse4" class="panel-collapse collapse">
                            <div class="panel-body">
                                <p>Douard, E., Zeribi, A., Schramm, C., Tamer, P., Loum, M. A., Nowak, S., Saci, Z., Lord, M.-P., Rodriguez-Herreros, B., Jean-Louis, M., Moreau, C., Loth, E., Schumann, G., Pausova, Z., Elsabbagh, M., Almasy, L., Glahn, D. C., Bourgeron, T., Labbe, A., … Jacquemont, S.</p>
                                
                                 <br>
                                <h4>Abstract</h4>
                                <p>Objective Deleterious copy number variants (CNVs) are identified in up to 20% of individuals with autism. However, only 13 genomic loci have been formally associated with autism because the majority of CNVs are too rare to perform individual association studies. To investigate the implication of undocumented CNVs in neurodevelopmental disorders, we recently developed a new framework to estimate their effect-size on intelligence quotient (IQ) and sought to extend this approach to autism susceptibility and multiple cognitive domains.<br> Methods We identified CNVs in two autism samples (Simons Simplex Collection and MSSNG) and two unselected populations (IMAGEN and Saguenay Youth Study). Statistical models integrating scores of genes encompassed in CNVs were used to explain their effect on autism susceptibility and multiple cognitive domains.<br> Results Among 9 scores of genes, the “probability-of-being loss-of-function intolerant” (pLI) best explains the effect of CNVs on IQ and autism risk. Deletions decrease IQ by a mean of 2.6 points per point of pLI. The effect of duplications on IQ is three-fold smaller. The odd ratios for autism increases when deleting or duplicating any point of pLI. This increased autism risk is similar in subgroups of individuals below or above median IQ. Once CNV effects on IQ are accounted for, autism susceptibility remains mostly unchanged for duplications but decreases for deletions. Model estimates for autism risk overlap with previously published observations. Deletions and duplications differentially affect social communication, behaviour, and phonological memory, whereas both equally affect motor skills. <br>Conclusions Autism risk conferred by duplications is less influenced by IQ compared to deletions. CNVs increase autism risk similarly in individuals with high and low IQ. Our model, trained on CNVs encompassing >4,500 genes, suggests highly polygenic properties of gene dosage with respect to autism risk. These models will help interpreting CNVs identified in the clinic.</p>
                            </div>
                        </div>
                    </div>
		
		    <div class="panel panel-default">
                    <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
                    <div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapse3">
                                    The genetics-BIDS extension: Easing the search for genetic data associated with human brain imaging.
                                </a>
                                
                                <h5> <a target="_blank" href="https://bids-specification.readthedocs.io/en/v1.3.0/04-modality-specific-files/08-genetic-descriptor.html">Giga Science 2020</a></h5>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.1093/gigascience/giaa104"></div>

                            </h4>
                        </div>
                         <div id="collapse3" class="panel-collapse collapse">
                            <div class="panel-body">
                                <p>Moreau, C., Martineau J.L., Blair, R., Markiewicz C, Turner J., Calhoun V., Nichols T, Pernet C.</p>
                                
                                 <br>
                                <h4>Abstract</h4>
                                <p>Metadata are key in our ability to search databases. Without them, researchers would spend hours examining datasets in the hope to find data with features they are interested in. Brain imaging genetics is at the intersection of two disciplines each one with dedicated dictionaries and ontologies facilitating data search and analysis. <br>Here we present the genetics brain imaging data structure extension: it consists of metadata files for human brain imaging data to which they are linked to and describe succinctly  the genomic and transcriptomic data associated to them, possibly in different databases. This extension will facilitate identifying micro-scale molecular features that are linked to macro-scale imaging repositories facilitating data aggregation across studies.</p>
                            </div>
                        </div>
                    </div>
                    </div>
			
		    <div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapseOne">
                                   Mutations associated with neuropsychiatric conditions delineate functional brain connectivity dimensions contributing to autism and schizophrenia
                                </a>
                                <h5> <a target="_blank" href="https://www.nature.com/articles/s41467-020-18997-2">Nature Communications 2020</a></h5>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.1101/862615"></div>
                            </h4>
                        </div>
			<div id="collapseOne" class="panel-collapse collapse in">

                            <div class="panel-body">
                                <p>Moreau, C., Urchs, S., Orban, P., Schramm, C., Dumas, G., Labbe, A., Huguet, G., Douard, E., Quirion, P.-O., Lin, A., Kushan, L., Grot, S., Luck, D., Mendrek, A., Potvin, S., Stip, E., Bourgeron, T., Evans, A. C., SimonsVIP Consortium, Bearden, CE., Bellec, P., Jacquemont, S.  </p>
                                
                                 <br>
                                <h4>Abstract</h4>

                                <p> 16p11.2 and 22q11.2 Copy Number Variants (CNVs) confer high risk for Autism Spectrum Disorder (ASD), schizophrenia (SZ), and Attention-Deficit-Hyperactivity-Disorder (ADHD), but their impact on functional connectivity (FC) networks remains unclear. <br>We analyzed resting-state functional magnetic resonance imaging data from 101 CNV carriers, 755 individuals with idiopathic ASD, SZ, or ADHD and 1,072 controls. We used CNV FC-signatures to identify major dimensions contributing to complex idiopathic conditions. <br>CNVs had large mirror effects on FC at the global and regional level, and their effect-sizes were twice as large as those of idiopathic conditions. Thalamus, somatomotor, and posterior insula regions played a critical role in dysconnectivity shared across deletions, duplications, idiopathic ASD, SZ but not ADHD. Individuals with higher similarity to deletion FC-signatures exhibited worse behavioral and cognitive symptoms. These seemingly distinct neuropsychiatric mutations showed similar gene co-expression patterns and converged on FC dimensions, that may represent mechanistic building blocks shared across idiopathic conditions.</p>
                            </div>
                        </div>
                    </div>
			
                    <div class="panel panel-default">
                    <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
                    <div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapse15">
                                    Dissecting autism and schizophrenia through neuroimaging genomics. 
                                </a>
                                
                                <h5> <a target="_blank" href="https://academic.oup.com/brain/advance-article/doi/10.1093/brain/awab096/6168126">Brain 2021</a></h5>
				 <div class='altmetric-embed' data-badge-type='4' data-doi="10.1093/brain/awab096"></div>
                            </h4>
                        </div>
			 <div id="collapse15" class="panel-collapse collapse in">

                            <div class="panel-body">
                                <p>Moreau, C., Raznahan, A., Bellec P.,, Chakravarty M., Thompson, P.M., Jacquemont, S.</p>
                                
                                 <br>
                                <h4>Abstract</h4>
                                <p>Top-down neuroimaging and genomic studies of autism spectrum disorder and schizophrenia have revealed intriguingly small neuroimaging effect-sizes and an extreme polygenic architecture. It is also evident that both genomic variants and neuroimaging patterns are shared across psychiatric diagnoses - suggesting pleiotropic mechanisms. However, it remains unknown if variants and patterns are related to core cognitive dimensions or to comorbidities. <br>Bottom-up studies start at the level of molecular factors to study mechanisms related to biological risk irrespective of clinical manifestations. Such approaches reveal that the effect-sizes of high-risk psychiatric mutations are equally large for neuroimaging, cognitive and behavioural traits in stark contrast to top-down studies. Low specificity has been perplexing with studies showing that broad classes of genomic variants affect a similar range of psychiatric and cognitive dimensions. <br> We propose a roadmap based on multivariate strategies to integrate genomics, transcriptomics, neuroimaging and phenotypic data to deconvolve mechanisms involved in psychiatric conditions. New profiling based on specific biological processes and expression scores has considerable potential for parsing the contribution of genes and mechanisms to dimensions underlying psychiatric conditions. Such approaches will improve mechanistically informed predictive modelling for diagnosis and treatment outcomes.</p>
                            </div>
                        </div>
                    </div>
                    </div>
			
		<div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapse20">
                                    1q21.1 distal copy number variants are associated with cerebral and cognitive alterations in humans. 
                                </a>
                                
                                <h5> <a target="_blank" href="https://www.nature.com/articles/s41398-021-01213-0">Translational Psychiatry 2021</a></h5>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.1038/s41398-021-01213-0"></div>

                            </h4>
                        </div>
                        <div id="collapse20" class="panel-collapse collapse">
                            <div class="panel-body">
                                <p>Sønderby I, van der Meer D., Moreau C.A., (...) Jacquemont S., Thompson P., Andreassen O., Bøen R.</p>
                                
                                 <br>
                                <h4>Abstract</h4>
				    <p>Low-frequency 1q21.1 distal deletion and duplication copy number variant (CNV) carriers are predisposed to multiple neurodevelopmental disorders including schizophrenia, autism and intellectual disability. Human carriers display a high prevalence of micro- and macrocephaly in deletion and duplication carriers, respectively. The underlying brain structural diversity remains largely unknown. We systematically called CNVs in 38 cohorts from the large-scale ENIGMA-CNV collaboration and the UK biobank and identified 28 1q21.1 distal deletion and 22 duplication carriers and 37,088 non-carriers (48 % male) derived from 15 distinct MRI scanner sites. With standardized methods, we compared subcortical and cortical brain measures (all) and cognitive performance (UK biobank only) between carrier groups also testing for mediation of brain structure on cognition. We identified positive dosage effects of copy number on intracranial volume (ICV) and total cortical surface area, with largest effects in frontal and cingulate cortices, and negative dosage effects on caudate and hippocampal volumes. The carriers displayed distinct cognitive deficit profiles in cognitive tasks from the UK biobank with intermediate decreases in duplication carriers and somewhat larger in deletion carriers – the latter potentially mediated by ICV or cortical surface area. These results shed light on pathobiological mechanisms of neurodevelopmental disorders, by demonstrating gene dose effect on specific brain structures and effect on cognitive function.</p>

				</div>
                        </div>
                    </div> 
			
			<div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapse30">
                                    Effects of copy number variations on brain structure and risk for psychiatric illness: Large-scale studies from the ENIGMA Working Groups on CNVs
                                </a>
                                
                                <h5> <a target="_blank" href="https://onlinelibrary.wiley.com/doi/10.1002/hbm.25354">Human Brain Mapping 2021</a></h5>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.1002/hbm.25354"></div>

                            </h4>
                        </div>
                        <div id="collapse30" class="panel-collapse collapse">
                            <div class="panel-body">
                                <p>Sønderby I, Ching C., (...) Moreau C.A., (...) Jacquemont S., Thompson P., Bearden C., Andreassen O.</p>
                                
                                 <br>
                                <h4>Abstract</h4>
				    <p>The Enhancing Neuroimaging Genetics through Meta-Analysis copy number variant (ENIGMA-CNV) and 22q11.2 Deletion Syndrome Working Groups (22q-ENIGMA WGs) were created to gain insight into the involvement of genetic factors in human brain development and related cognitive, psychiatric and behavioral manifestations. To that end, the ENIGMA-CNV WG has collated CNV and magnetic resonance imaging (MRI) data from ~49,000 individuals across 38 global research sites, yielding one of the largest studies to date on the effects of CNVs on brain structures in the general population. The 22q-ENIGMA WG includes 12 international research centers that assessed over 533 individuals with a confirmed 22q11.2 deletion syndrome, 40 with 22q11.2 duplications, and 333 typically developing controls, creating the largest-ever 22q11.2 CNV neuroimaging data set. In this review, we outline the ENIGMA infrastructure and procedures for multi-site analysis of CNVs and MRI data. So far, ENIGMA has identified effects of the 22q11.2, 16p11.2 distal, 15q11.2, and 1q21.1 distal CNVs on subcortical and cortical brain structures. Each CNV is associated with differences in cognitive, neurodevelopmental and neuropsychiatric traits, with characteristic patterns of brain structural abnormalities. Evidence of gene-dosage effects on distinct brain regions also emerged, providing further insight into genotype-phenotype relationships. Taken together, these results offer a more comprehensive picture of molecular mechanisms involved in typical and atypical brain development. This ‘genotype-first’ approach also contributes to our understanding of the etiopathogenesis of brain disorders. Finally, we outline future directions to better understand effects of CNVs on brain structure and behavior.</p>

				</div>
                        </div>
                    </div> 
		
			
		<div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapse31">
					The Open Brain Consent: Informing research participants and obtaining consent to share brain imaging data
                                </a>
                                
                                <h5> <a target="_blank" href="https://onlinelibrary.wiley.com/doi/10.1002/hbm.25351">Human Brain Mapping 2021</a></h5>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.1002/hbm.25351"></div>

                            </h4>
                        </div>
                        <div id="collapse31" class="panel-collapse collapse">
                            <div class="panel-body">
                                <p>Banier E., (...), Moreau C.A., (...) Pernet C.</p>
                                
                                 <br>
                                <h4>Abstract</h4>
				    <p>Having the means to share research data openly is essential to modern science. For human research, a key aspect in this endeavour is obtaining consent from participants, not just to take part in a study, which is a basic ethical principle, but also to share their data with the scientific community. To ensure that the participants’ privacy is respected, national and/or supranational regulations and laws are in place. It is, however, not always clear to researchers what the implications of those are, nor how to comply with them. The Open Brain Consent (https://open-brain-consent.readthedocs.io) is an international initiative that aims to provide researchers in the brain imaging community with information about data sharing options and tools. We present here a short history of this project and its latest developments, and share pointers to consent forms, including a template consent form that is compliant with the EU General Data Protection Regulation. We also share pointers to an associated data user agreement that is not only useful in the EU context, but also for any researchers dealing with personal (clinical) data elsewhere.</p>
				</div>
                        </div>
                    </div> 
			
                    
                    <div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapse16">
                                    Effects of eight neuropsychiatric copy number variants on human brain structure. 
                                </a>
                                
                                <h5> <a target="_blank" href="https://doi.org/10.1038/s41398-021-01490-9">Translational Psychiatry 2021</a></h5>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.1038/s41398-021-01490-9"></div>

                            </h4>
                        </div>
                        <div id="collapse16" class="panel-collapse collapse">
                            <div class="panel-body">
                                <p>Modenato, C., Kumar, K., Moreau, C.A., Martin-Brevet, S., Huguet, G., Schramm, C., Martineau, J.-L., Martin, C.-O., Younis, N., Tamer, P., Douard, E. A., Thebault-Dagher, F., Cote, V., Charlebois, A.-R., Deguire, F., Maillard, A. M., Rodriguez-Herreros, B., Pain, A., Richetin, S., … Jacquemont.</p>
                                
                                 <br>
                                <h4>Abstract</h4>
                                <p>Background: Copy Number Variants (CNVs) associated with autism and schizophrenia have large effects on brain anatomy. Yet, neuroimaging studies have been conducted one mutation at a time. We hypothesize that neuropsychiatric CNVs may exert general effects on brain morphometry because they confer risk for overlapping psychiatric conditions. <br>Methods: We analyzed T1-weighted MRIs and characterized shared patterns on brain anatomy across 8 neuropsychiatric CNVs. Clinically ascertained samples included 1q21.1 (n=48), 16p11.2 (n=156), or 22q11.2 (n=96) and 331 non-carriers. Non-clinically ascertained samples from the UK Biobank included 1q21.1 (n=19), 16p11.2 (n=8), 22q11.2 (n=9), 15q11.2 (n=148) and 965 non-carriers. Canonical correlation analysis (CCA) and univariate models were used to interrogate brain morphometry changes across 8 CNVs. <br>Results: Eight CNVs affect regional brain volumes along two main gene-morphometry dimensions identified by CCA. While fronto-temporal regions contributed to dimension 1, dimension 2 was driven by subcortical, parietal and occipital regions. Consistently, voxel-wise whole-brain analyses identified the same regions involved in patterns of alteration present across the 4 deletions and duplications. These neuroanatomical patterns are similar to those observed in cross-psychiatric disorder meta-analyses. Deletions and duplications at all 4 loci show mirror effects at either the global and/or the regional level. <br>Conclusion: Neuropsychiatric CNVs share neuroanatomical signatures characterized by a parsimonious set of brain dimensions. The latter may underlie the risk conferred by CNVs for a similar spectrum of neuropsychiatric conditions.</p>
                            </div>
                        </div>
                    </div>
                    
                    <div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapse5">
                                    Subtypes of functional connectivity associate robustly with ASD diagnosis. 
                                </a>
                                
                                <h5> <a target="_blank" href="https://doi.org/10.1101/2020.04.14.040576">bioRxiv</a></h5>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.1101/2020.04.14.040576"></div>

                            </h4>
                        </div>
                        <div id="collapse5" class="panel-collapse collapse">
                            <div class="panel-body">
                                <p>Urchs, S. G. W., Tam, A., Orban, P., Moreau, C., Benhajali, Y., Nguyen, H. D., Evans, A. C., Bellec, P.</p>
                                
                                 <br>
                                <h4>Abstract</h4>
                                <p>Our understanding of the changes in functional brain organization in autism is hampered by the extensive heterogeneity that characterizes this neurodevelopmental disorder. Data driven clustering offers a straightforward way to decompose this heterogeneity into subtypes of distinguishable connectivity types and promises an unbiased framework to investigate behavioural symptoms and causative genetic factors. Yet the robustness and generalizability of these imaging subtypes is unknown. <br>Here, we show that unsupervised functional connectivity subtypes are moderately associated with the clinical diagnosis of autism, and that these associations generalize to independent replication data. We found that subtypes identified robust patterns of functional connectivity, but that a discrete assignment of individuals to these subtypes was not supported by the data. <br>Our results support the use of data driven subtyping as a data dimensionality reduction technique, rather than to establish clinical categories.</p>
                            </div>
                        </div>
                    </div>    
		
		
			
			<div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapse21">
                                    Structural and functional brain alterations revealed by neuroimaging in CNV carriers. 
                                </a>
                                
                                <h5> <a target="_blank" href="https://www.sciencedirect.com/science/article/pii/S0959437X21000381">Current Opinion in Genetics & Development</a> 2021 </h5>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.1016/j.gde.2021.03.002"></div>

                            </h4>
                        </div>
                        <div id="collapse21" class="panel-collapse collapse">
                            <div class="panel-body">
                                <p>Moreau, C.A., Ching, C.R.K, Kumar, K., Jacquemont, S., Bearden C.E.</p>
                                
                                 <br>
                                <h4>Abstract</h4>
				    <p>Copy Number Variants (CNVs) are associated with elevated rates of neuropsychiatric disorders. A ‘genetics-first’ approach, involving the CNV effects on the brain, irrespective of clinical symptomatology, allows investigation of mechanisms underlying neuropsychiatric disorders in the general population. Recent years have seen an increasing number of larger multisite neuroimaging studies investigating the effect of CNVs on structural and functional brain endophenotypes. Alterations overlap with those found in idiopathic psychiatric conditions but effect sizes are 2 to 5-fold larger. Here we review new CNV-associated structural and functional brain alterations and outline the future of neuroimaging genomics research with particular emphasis on developing new resources for the study of high-risk CNVs and rare genomic variants.</p>
                            </div>
                        </div>
                    </div> 	
                                        

                   <div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapse33">
					Subtypes of brain activation are heritable and genetically linked with behavior in the Human Connectome Project sample
                                </a>
                                
                                <h5> <a target="_blank" href="https://psyarxiv.com/excdq/">PsyArXiv</a></h5>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.31234/osf.io/excdq"></div>

                            </h4>
                        </div>
                        <div id="collapse33" class="panel-collapse collapse">
                            <div class="panel-body">
                                <p>Benhajali Y, Badhwar A., Urchs S., Moreau C., Chouinard-Decorte F., Vainik U., Ferré P., Orban P., Pérusse D, Bellec P.</p>
                                
                                 <br>
                                <h4>Abstract</h4>
				    <p>Many imaging and genetics studies have aimed to clarify whether the brain acts as an intermediate phenotype mediating the influence of genes in human behaviour. Brain activations in response to task demands are heterogeneous at the individual level, but also follow common patterns at the group level. Some studies have addressed this tension between heterogeneity and homogeneity by identifying groups of individuals that share the same brain activations patterns, called brain activation subtypes. In this work, we aimed to assess the viability of brain subtypes as endophenotypes intermediate between genes and behavior. We extracted brain activation subtypes separately for seven fMRI tasks, in 842 participants from the Human Connectome Project (HCP). We estimated the heritability of these subtypes and their genetic correlation with behavioral measures obtained inside and outside the scanner. Across all tasks, subtypes ranged from a predominantly ‘deactivating’ pattern towards a more ‘activating’ pattern of brain activity, with a heritability estimate ranging from 0 to 0.62. We observed high genetic and phenotypic correlation between behavioral measures and brain activation subtypes only for language and working memory tasks. Our results showed a significant genetic grounding of brain activation subtypes and they appear as a simple yet effective technique to tackle heterogeneity into imaging genetics studies.</p>
				</div>
                        </div>
                    </div> 
                      
                   <div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapse34">
					Lessons learnt from neuroimaging studies of Copy Number Variants, a systematic review
                                </a>
                                
                                <h5> <a target="_blank" href="https://www.biologicalpsychiatryjournal.com/article/S0006-3223(21)01394-9/abstract">Biological Psychiatry</a> 2021</h5>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.1016/j.biopsych.2021.05.028"></div>

                            </h4>
                        </div>
                        <div id="collapse34" class="panel-collapse collapse">
                            <div class="panel-body">
                                <p> Modenato C*, Martin-Brevet S*, Moreau C.A., Rodriguez-Herreros B., Kumar K., Draganski B., Sønderby I.E., Jacquemont S. </p>
                                
                                 <br>
                                <h4>Abstract</h4>
				    <p>Pathogenic Copy Number Variants (CNVs) and aneuploidies are associated with autism spectrum disorder, schizophrenia, and are identified in up to 15% of patients referred for neuropsychiatric symptoms.
Brain mechanisms mediating neuropsychiatric risk remain largely unknown, but there is a steady increase in computational anatomy publications investigating CNVs. Studies have been conducted one mutation at a time, leaving the field with a complex catalog of brain alterations linked to different genomic loci. The latter has hindered attempts to understand general principles linking CNVs to brain architecture.
Our aim is to provide a general framework to understand the effects on neuroimaging phenotypes across multiple genomic loci. We conducted a systematic review of 64 computational anatomy studies on 16 CNVs and chromosomal aneuploidies (reciprocal 15q11.2, 22q11.2, 1q21.1 distal, 16p11.2 distal and proximal, 7q11.23 deletions and chromosomes, X, Y and 21).
Large to moderate effect sizes on global and regional brain morphometry are observed across all neuropsychiatric CNVs, which is in stark contrast with the milder effects observed in idiopathic psychiatric disorders. Effect sizes at the regional level are in line with the symptom severity of CNVs. Data also suggest independent CNV effects on global versus regional measures as well as on surface versus thickness. Multivariate analyses suggest shared variance of morphometry alterations across multiple CNVs.
This review highlights that to date, neuroimaging studies have only been conducted in a minute proportion of CNVs identified in the clinic and we propose strategies to transition from single variants to genome-wide studies.</p> 
				</div>                       
			   </div>
                    </div>
			
                   <div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapse44">
					Interindividual Differences in Cortical Thickness and Their Genomic Underpinnings in Autism Spectrum Disorder
                                </a>
                                
                                <h5> <a target="_blank" href="https://ajp.psychiatryonline.org/doi/full/10.1176/appi.ajp.2021.20050630">American Journal of Psychiatry</a> 2021</h5>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.1176/appi.ajp.2021.20050630"></div>

                            </h4>
                        </div>
                        <div id="collapse44" class="panel-collapse collapse">
                            <div class="panel-body">
                                <p> Ecker C, Pretzch C, Bletsch A, … Moreau C, …, Bourgeron T, Beckmann C., the EU-AIMS LEAP Group, Murphy D.</p>
                                
                                 <br>
                                <h4>Abstract</h4>
				    <p>
				    Autism spectrum disorder (ASD) is accompanied by highly individualized neuroanatomical deviations that potentially map onto distinct genotypes and clinical phenotypes. This study aimed to link differences in brain anatomy to specific biological pathways to pave the way toward targeted therapeutic interventions. 
				    The authors examined neurodevelopmental differences in cortical thickness and their genomic underpinnings in a large and clinically diverse sample of 360 individuals with ASD and 279 typically developing control subjects (ages 6–30 years) within the EU-AIMS Longitudinal European Autism Project (LEAP). The authors also examined neurodevelopmental differences and their potential pathophysiological mechanisms between clinical ASD subgroups that differed in the severity and pattern of sensory features. In addition to significant between-group differences in “core” ASD brain regions (i.e., fronto-temporal and cingulate regions), individuals with ASD manifested as neuroanatomical outliers within the neurotypical cortical thickness range in a wider neural system, which was enriched for genes known to be implicated in ASD on the genetic and/or transcriptomic level. Within these regions, the individuals’ total (i.e., accumulated) degree of neuroanatomical atypicality was significantly correlated with higher polygenic scores for ASD and other psychiatric conditions, and it scaled with measures of symptom severity. Differences in cortical thickness deviations were also associated with distinct sensory subgroups, especially in brain regions expressing genes involved in excitatory rather than inhibitory neurotransmission. The study findings corroborate the link between macroscopic differences in brain anatomy and the molecular mechanisms underpinning heterogeneity in ASD, and provide future targets for stratification and subtyping.
				    </p> 
				</div>                       
			   </div>
                    </div>
                  
		    <div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapse45">
                                    Genetic heterogeneity shapes brain connectivity in psychiatry. 
                                </a>
                                
                                <h5> <a target="_blank" href="https://www.sciencedirect.com/science/article/pii/S0006322322015529?via%3Dihub">Biological Psychiatry 2022</a></h5>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.1016/j.biopsych.2022.08.024"></div>

                            </h4>
                        </div>
                        <div id="collapse45" class="panel-collapse collapse in">
                            <div class="panel-body">
                                <p> Moreau CA, Harvey A, Kumar K, Huguet G, Urchs S, Douard EA, ..., Thompson, P.M., Bourgeron T., Bellec, P., Jacquemont, S.</p>
                              
                                 <br>
                                <h4>Abstract</h4>
				    <p>Background</p> <br><p>
Polygenicity and genetic heterogeneity pose great challenges for studying psychiatric conditions. Genetically-informed approaches have been implemented in neuroimaging studies to address this issue. However, the effects on functional connectivity of rare and common genetic risks for psychiatric disorders are largely unknown. Our objectives were to estimate and compare the effect-sizes on brain connectivity of psychiatric genomic risk factors with various levels of complexity: oligo-, multi-genic copy number variants (CNVs), and polygenic risk scores (PRS) as well as idiopathic psychiatric conditions and traits.

Methods</p> <br><p>
Resting-state functional-MRI data were processed using the same pipeline across nine datasets. Twenty-nine connectome-wide association studies were performed to characterize the effects of 15 CNVs (1003 carriers), 7 PRS, 4 idiopathic psychiatric conditions (1022 individuals with either autism, schizophrenia, bipolar conditions, or ADHD), and 2 traits (31424 unaffected controls).

Results</p> <br><p>
Effect sizes on connectivity were largest for psychiatric CNVs (estimates: 0.2 to 0.65 z-score) followed by psychiatric conditions (0.15 to 0.42), neuroticism and fluid intelligence (0.02 to 0.03), and PRS (0.01 to 0.02). Effect-sizes of CNVs on connectivity were correlated to their effects on cognition and risk for disease (r=0.9, p=5.93e-06). However, effect sizes of CNVs adjusted for the number of genes significantly decreased from small oligogenic to large multigenic CNVs (r=-0.88, p=8.78e-06). PRS had disproportionately low effect sizes on connectivity compared to CNVs conferring similar risk for disease.

Conclusion</p> <br><p>
Heterogeneity and polygenicity impact our ability to detect brain connectivity alterations underlying psychiatric manifestations.</p></div>
                    </div>
			    
		<div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapse23">
                                    Brain functional connectivity mirrors genetic pleiotropy in psychiatric conditions. 
                                </a>
                                
                                <h5> <a target="_blank" href="https://academic.oup.com/brain/advance-article/doi/10.1093/brain/awac315/6691797">Brain 2022</a></h5>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.1093/brain/awac315"></div>

                            </h4>
                        </div>
                        <div id="collapse23" class="panel-collapse collapse in">
                            <div class="panel-body">
                                <p> Moreau CA, Kumar K, Harvey A, Huguet G, Urchs S, Schultz LM, ..., Thompson, P.M., Bourgeron T., Bellec, P., Jacquemont, S.</p>
                              
                                 <br>
                                <h4>Abstract</h4>
				    <p>Pleiotropy occurs when a genetic variant influences more than one trait. This is a key property of the genomic architecture of psychiatric disorders and has been observed for rare and common genomic variants. It is reasonable to hypothesize that the microscale genetic overlap (pleiotropy) across psychiatric conditions and cognitive traits may lead to similar overlaps at the macroscale brain level such as large-scale brain functional networks.
We took advantage of brain connectivity, measured by resting-state functional MRI to measure the effects of pleiotropy on large-scale brain networks, a putative step from genes to behavior. We processed nine resting-state functional MRI datasets including 32,726 individuals and computed connectome-wide profiles of seven neuropsychiatric copy-number-variants, five polygenic scores, neuroticism, and fluid intelligence as well as four idiopathic psychiatric conditions.
Nine out of nineteen pairs of conditions and traits showed significant functional connectivity correlations (rFunctional connectivity), which could be explained by previously published levels of genomic (rGenetic) and transcriptomic (rTranscriptomic) correlations with moderate to high concordance: rGenetic - rFunctional connectivity= 0.71 [0.40-0.87] and rTranscriptomic - rFunctional connectivity= 0.83 [0.52; 0.94]. Extending this analysis to functional connectivity profiles associated with rare and common genetic risk showed that 30 out of 136 pairs of connectivity profiles were correlated above chance. These similarities between genetic risks and psychiatric disorders at the connectivity level were mainly driven by the overconnectivity of the thalamus and the somatomotor networks. Our findings suggest a substantial genetic component for shared connectivity profiles across conditions and traits, opening avenues to delineate general mechanisms - amenable to intervention - across psychiatric conditions and genetic risks.
				</p></div>
                    </div>

		   
        
                    </div>
                </div>
                <!--/#accordion-->
            </div>
            <!--/#accordion-container-->
            
           <div id="carousel-container">
                <h2 class="page-header">Galerie</h2>
                <div id="carousel-example-generic" class="carousel slide" data-ride="carousel">
                    <ol class="carousel-indicators visible-xs">
                        <li data-target="#carousel-example-generic" data-slide-to="0" class="active"></li>
                        <li data-target="#carousel-example-generic" data-slide-to="1"></li>
                        <li data-target="#carousel-example-generic" data-slide-to="2"></li>
                        <li data-target="#carousel-example-generic" data-slide-to="3"></li>
                        <li data-target="#carousel-example-generic" data-slide-to="4"></li>
                        <li data-target="#carousel-example-generic" data-slide-to="5"></li>
                        <li data-target="#carousel-example-generic" data-slide-to="6"></li>
                        <li data-target="#carousel-example-generic" data-slide-to="7"></li>
                    </ol>
                   
                    <div class="carousel-inner">

                        <div class="item active"> 
                        <iframe width="1140" height="487" src="https://www.youtube.com/embed/P6450HqNKyo" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
                        </div>
                        <div class="item"> 
                        <iframe width="1140" height="487" src="https://www.youtube.com/embed/ngh-l2-qXDU?start=56" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
                        </div>
                        <div class="item"> 
                        <iframe width="1140" height="487" src="https://www.youtube.com/embed/fs4-xMb-4Kc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
                        </div>
                         <div class="item">
                        <img src="images/cm.jpg" alt="">
                        </div>
                        
                    </div>
                    
                    <a class="left carousel-control hidden-xs" href="#carousel-example-generic" data-slide="prev">
                        <span class="glyphicon glyphicon-chevron-left"></span>
                    </a>
                    <a class="right carousel-control hidden-xs" href="#carousel-example-generic" data-slide="next">
                        <span class="glyphicon glyphicon-chevron-right"></span>
                    </a>
                </div><!--/#carousel-example-generic-->
            </div>
            <!--/#carousel-container-->
                    </div>
        </div>
</section>
    <section id="services">
       
        <div class="container">
            <div class="row">
             <h2 class="page-header"> News - Media</h2>
                                
                <div class="col-sm-6 text-center padding wow fadeIn">
                    <div class="single-service">

                        <iframe width="500" height="500" src="https://www.pathlms.com/ohbm/courses/12238/sections/15843/video_presentations/138313" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
                    </div>
                </div>



                 <div class="col-sm-6 text-center padding wow fadeIn" >
                   <div class="single-service">

                        <a class="twitter-timeline" data-lang="en" data-width="500" data-height="500" href="https://twitter.com/ClaraMoreau9?ref_src=twsrc%5Etfw">Tweets by ClaraMoreau9</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
             </div>
            </div>
        </div>
    </section>
    
    <section id="services"> 
         <div class="container">
            <div class="row">
             <h2 class="page-header"> Formal CV </h2> <p align=middle><a href="/ClaraMOREAU_CV.pdf" download> <i>Pdf to download</i></a></p>

            <iframe src="/ClaraMOREAU_CV.pdf" width="100%" height="400px" align=middle></iframe>
                
            </div>
        </div>
    </section>
        
    <section id="services">
       
        <div class="container">
            <div class="row">
             <h2 class="page-header"> Contact <div class="social-icons pull-right">
                        <ul class="nav nav-pills">
                            <li><a href="https://twitter.com/ClaraMoreau9"><i class="fa fa-twitter"></i></a></li>
                            <li><a href="https://scholar.google.fr/citations?user=tkN9j_cAAAAJ&hl=fr&oi=ao"><i class="ai ai-google-scholar-square ai-1x"></i></a></li>
                            <li><a href="https://ca.linkedin.com/in/clara-moreau-b44b6858"><i class="fa fa-linkedin"></i></a></li>
                            <li><a href="https://www.facebook.com/clara.moreau.94"><i class="fa fa-facebook"></i></a></li>
                        </ul>
            </div></h2>
                
                <div class="slide-text">

                        <p><img src="images/violon.png" align=right><b>E-mail</b> <a href="mailto:clara.moreau@pasteur.fr">clara.moreau@pasteur.fr</a><br>
                        <b>E-mail</b> <a href="mailto:clara.moreau@umontreal.ca">clara.moreau@umontreal.ca</a><br> 
                        <b>Phone</b> +1 (438) 763 6198<br> Montreal, QC Canada </p> 
                    
                </div> 
                <div class="copyright-text text-align">
                        <h5 align=middle>&copy; 2021.</h5>
                        <h6 align=middle>Adapted by CMoreau from <a target="_blank" href="http://www.themeum.com">Themeum</a></h6>
                    </div>
                                
               
             </div>
            </div>
        
    </section>
      

    
      </section>

                                    
    <!--/#clients-->
    <script type="text/javascript" src="js/jquery.js"></script>
    <script type="text/javascript" src="js/bootstrap.min.js"></script>
    <script type="text/javascript" src="js/lightbox.min.js"></script>
    <script type="text/javascript" src="js/wow.min.js"></script>
    <script type="text/javascript" src="js/main.js"></script>   
</body>
</html>
