<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="">
    <meta name="author" content="">
    <title>Home | Moreau Clara </title>
    <link href="css/bootstrap.min.css" rel="stylesheet">
    <link href="css/font-awesome.min.css" rel="stylesheet">
    <link href="css/animate.min.css" rel="stylesheet"> 
    <link href="css/lightbox.css" rel="stylesheet"> 
	<link href="css/main.css" rel="stylesheet">
	<link href="css/responsive.css" rel="stylesheet">

    <!--[if lt IE 9]>
	    <script src="js/html5shiv.js"></script>
	    <script src="js/respond.min.js"></script>
        http://www.free-css.com/free-css-templates/page204/triangle
    <![endif]-->       
    <link rel="shortcut icon" href="images/ico/favicon.ico">
    <link rel="stylesheet" href="css/academicons.min.css">

   </head><!--/head-->

<body>
	<header id="header">      
        <div class="container">
            <div class="row">
                <div class="col-sm-20 overflow">
                   <div class="social-icons pull-right">
                        <ul class="nav nav-pills">
                            <li><a href="https://twitter.com/ClaraMoreau9"><i class="fa fa-twitter"></i></a></li>
                            <li><a href="https://scholar.google.fr/citations?user=tkN9j_cAAAAJ&hl=fr&oi=ao"><i class="ai ai-google-scholar-square ai-1x"></i></a></li>
                            <li><a href="https://ca.linkedin.com/in/clara-moreau-b44b6858"><i class="fa fa-linkedin"></i></a></li>
                            <li><a href="https://www.facebook.com/clara.moreau.94"><i class="fa fa-facebook"></i></a></li>
                        </ul>
                    </div> 
                                    <div class="slide-text">
                        <h1>Neuroscience, Psychiatry and Genomics</h1>
                        <h2>Clara Moreau, Personal website</h2>                       

                </div>
                </div>
             </div>
        </div>
           </header>
    <!--/#header-->
    
    <section id="home-slider">
        <div class="container">
            <div class="row">
                <div class="main-slider">
                    <img src="images/home/slider/sun.png" class="slider-sun" alt="slider image">
                    <img src="images/home/slider/birds1.png" class="slider-birds1" alt="slider image">
                </div>
            </div>
        </div>
        <div class="preloader"><i class="fa fa-sun-o fa-spin"></i></div>
    </section>
    <!--/#home-slider-->
    
    
    <section id="services">
        <div class="container">
            <div class="row">
                <div id="tab-container">
                <h2 class="page-header">Curriculum vitae</h2>
                    <div class="col-md-6">
                        <ul id="tab" class="nav nav-tabs">
                            <li class="active"><a href="#tab1-item1" data-toggle="tab">Education <img src="images/academic.png"></a></li>
                            <li><a href="#tab1-item2" data-toggle="tab">Workshop <img src="images/workshop.png"></a></li>
                            <li><a href="#tab1-item3" data-toggle="tab">Teaching <img src="images/teaching.png"></a></li>

                        </ul>
                        <div class="tab-content">
                            <div class="tab-pane fade active in" id="tab1-item1">
                                <li>2020-2022: Postdoctoral, Imaging Genetic Center, University of South California <img src="images/usa.png"></li>
                                <p>(Coming) Advisors: P.M. Thompson<br>
                                <li>2015-2020: PhD Candidate, Neurosciences Department, University of Montreal <img src="images/ca.png"></li>
                                <p>Mapping genome-wide neuropsychiatric mutation effects on functional brain connectivity: Copy number variants delineate dimensions contributing to autism and schizophrenia.<br>
                                Co-supervised by: Sébastien Jacquemont (Geneticist, Sainte Justine Hospital) and Pierre Bellec (Computer Science Department, SIMEXP lab, CR-IUGM, University of Montreal)</p>
                                <li>2019: Internship (2 months) Imaging Genetic Center <img src="images/usa.png"></li>
                                <p>Advisors: P.M. Thompson, University of South California<br>
                                Skills: DTI analyses (ENIGMA) - Cross-CNVs data</p>
                                <li>2014-2015: Research Assistant, Medical Genetics Department, CHUV, <img src="images/ch.png"></li>
                                <p>Skills: MRI Protocol development and Scanning, Analyses (R, Matlab), Website development for family recruitment, neuropsychological assessment. S. Jacquemont’s laboratory</p>
                                <HR size=1 align=center width="80%">
                                <li>2012-2014: Master degree in Cognitive Sciences - Cogmaster <img src="images/FR.png"></li>
                                <p>Ecole Normale Supérieure and Descartes University, Paris</p>
                                <li>2013-2014 Internship Neurospin Institute, CEA <img src="images/FR.png"></li>
                                <p>Advisors: Marion Noulhiane and Lucie Hertz-Pannier (UNIACT)<br>
                                Skills: fMRI analysis (SPM, Matlab), Neurodevelopmental cohort, memory tasks (6 months).</p>
                                <li>2013 Internship (2 months),  University of Montreal <img src="images/ca.png"></li> 
                                <p>Advisor: Pr. P. Jolicoeur, Neuroscience department<br>
                                Skills: MEG/EEG analyses for an auditory task.</p>
                                <li>2012-2013 Internship, INSERM – UMR 663 Paris, France</li>
                                <p>Advisors: Pr. M. Noulhiane and Dr C. Chiron, Necker Children Hospital<br>
                                <HR size=1 align=center width="80%">
                                <li>2009-2012: BSc degree in Neuropsychology <img src="images/FR.png"></li>
                                <p>Descartes University, Paris-V </p>
                                <li>2012 Internship (summer), Lyon, France </li>
                                <p>Advisor: Pr. N. Ravel and Pr. Rémi Gervais <br> 
                                Skills: Recording olfactory cells in mice (electrophysiology) and signal processing</p>        
                                <HR size=1 align=center width="80%">

                                <li>2006-2009: Scientific Baccalaureate <img src="images/FR.png"></li>
                                <p>Victor Duruy High School, Paris 75007 France</p>  
                            </div>
                            <div class="tab-pane fade" id="tab1-item2">
                                <li>08.2017: Deep Learning - Summer school at the University of Montreal (MILA), Canada</li>
                                <li>06.2017: Workshop Brainhack - University of British Columbia, Canada</li>
                                <li>08.2016: Brain Imaging Genetics for Imagers, Summer school: Radboud University, Nijmegen</li>
                                <li>06.2016: Workshop Brainhack (fMRI) - Lausanne, Switzerland</li>
                                <li>02.2016: Workshop Brainhack (MRI) - Pasteur Institute, FR</li>
                                <li>04.2015: HackTheBrain workshop (EEG) - London, UK</li>
                                <li>08.2013: Basic and Advanced functional MRI, Summer school at McGill, Montreal, CA</li>
                                     
                            </div>
                            <div class="tab-pane fade" id="tab1-item3">
                                <li>06.2019-06.2020: Supervisor of A. Proulx (Honours degree, Neurosciences, UdeM)</li>
                                <li>2018 and 2019: Teaching Assistant, “Brain Imaging Techniques” (100 students, UdeM,  Pr. P. Bellec)</li>
                                <li>01.2019-07.2019: Supervisors of G. Dumais (BSc–3rd year, Neurosciences, UdeM)</li>
                                <li>05.2018-06.2018: Instructor at the Brainhack School 2018 (Imaging genetics, UdeM)</li>
                                <li>05.2016-08.2016: Supervisors of A. Casgrain-Cyr (BSc–3rd year, Bioinformatics, UdeM)</li>
                            </div>
                            
                        </div>
                    </div>
                     <div class="col-md-6">
                        <ul id="tab" class="nav nav-tabs">
                            <li class="active"><a href="#tab2-item1" data-toggle="tab">International conferences <img src="images/conf.png"></a></li>
                            <li><a href="#tab2-item2" data-toggle="tab">Science Dissemination <img src="images/transfer.png"></a></li>
                            <li><a href="#tab2-item3" data-toggle="tab">Other skills</a></li>
                        </ul>
                    
                        <div class="tab-content">
                            <div class="tab-pane fade active in" id="tab2-item1">
                                <li>06.2020 (coming) Human Brain Mapping</li>
                                <p>Symposium “Neuropsychiatric genetic variation shapes brain architecture by modulating gene expression”</p>   
                                <li>03.2020: Feindel BIC Lecture McGill University, Canada</li>
                                <p><a href="https://www.mcgill.ca/neuro/channels/event/feindel-virtual-brain-and-mind-seminar-neuro-mapping-neuropsychiatric-mutation-effects-functional-321797">Mapping neuropsychiatric mutation effects on functional brain connectivity to delineate dimensions contributing to autism and schizophrenia</a></p>   
                                <li>08.2019: Imaging Genetic Center, USC, USA</li>
                                <p><a href="https://www.facebook.com/photo.php?fbid=10156383123046892&set=pb.664156891.-2207520000..&type=3">High-risk psychiatric mutations modulate functional brain connectivity pointing to dimensions involved in autism and schizophrenia</a></p>
                                <li>06.2019 Human Brain Mapping, Rome</li>
                                <p>Symposium <a href="https://www.pathlms.com/ohbm/courses/12238/sections/15843/video_presentations/138313">A tough nut to crack: neurodevelopmental connectopathies</a></p>  
                                <li>05.2019 Society of Biological Psychiatry, Chicago</li>
                                <p>Symposium “Large Scale Imaging Studies of Rare Copy Number variants: Brain Imaging from Enigma and Other Large-Scale International Studies”</p>
                                <li>04.2019 International Society for Autism Research, Montreal</li>
                                <p>Symposium: “Human and Animal Models: Impact of High-Risk Copy Number Variants on Brain Structure, Functional Connectivity, and Sexual Development.”</p>        
                                <li>05.2016: University of Montreal, Canada (45 min)</li>
                                <p>"From the first human genome to recent application in genome edition: ethical issues.”</p>          
                                  <li>11.2014: EPFL, Switzerland</li>
                                <p>Symposium <a href="https://www.epfl-innovationpark.ch/epfled2015/protecting-ideas-liberating-innovation/">Protecting Ideas, Liberating Innovation, and Open collaboration</a></p>              
                                <li>09.2014: University of Lausanne, Switzerland</li>
                                <p>Symposium <a href="https://www.youtube.com/watch?v=fs4-xMb-4Kc&feature=youtu.be">Open Access, Open Data and Open Science: reality, and amalgam</a></p> 
                                <li>03.2014: Cité des Sciences et de l’Industrie, Paris, France</li>
                                <p>Symposium Conciliate Research, Innovation and Open Science?</p> 
                                <li>03.2014: Annual Cognitive science forum</li>
                                <p>Symposium<a href="https://www.youtube.com/watch?v=CHJVMPAQcXo">Artificial, animal, and human cognitive processes: How could we delineate Intelligence?</a></p> 
                            </div>
                            
                            <div class="tab-pane fade" id="tab2-item2">
                                <li>2019: BIDS contributor <a href="https://bids-specification.readthedocs.io/en/latest/06-extensions.html"> Brain Imaging Data Structure)</a></li> 
                                <p>Genomic information (C. Pernet).</p>
                                <li>06.2016: <a href="http://www.brainhack.org/OHBM2016/">Brain imaging workshop co-organizer</a> (Brainhack) </li> 
                                <p> with P. Bellec, Lausanne, Switzerland</p>
                                <li>2016: Part of the science dissemination team</li>
                                <p><a href="https://recherche.chusj.org/fr/Blogue/Clara-Moreau">Brain and Development research</a> section at the Ste Justine Hospital, Montreal</p>
                                <li>2015-2016: Radio show</li>
                                <p><a href="http://www.podcastscience.fm/emission/2015/01/20/podcast-science-202-les-maladies-mentales">Mental Ilnesses</a> <br> <a href="https://www.podcastscience.fm/emission/2013/11/24/podcast-science-152-dans-le-cerveau/">Inside the Brain (Neuroimaging techniques)</a></p>
                                <li>2014-2015: “Startup Weekend” workshops organizer and teacher</li>
                                <p>For PhD students, 6 editions <br> 
                                    Lausanne (EPFL), and Paris (ENS, CogInnov, and ESPCI). </p>
                                <li>2014-2015: Board member and web manager for “Science en marche”</li>
                                <p>National political committee <br> French organization of researchers</p>
                                <li>2014: Conference co-organizer at the ENS University of Paris</li>
                                <p>“Conciliate Open Science, Patents, and Intellectual property issues?”</p>
                                <li>2014-2015: Co-founder of Artificial intelligence lab at La Paillasse</li>
                                <p>Citizen lab, Paris</p>
                                <li>06.2015: Open innovation “Lift Conference”, Shanghai, China</li>
                                <li>03.2013: Co-organizer of the annual french Cognitive sciences Forum</li>
                                <p>Consciousness process of intelligence across the animal, the human, and the computer</p>
                                <li>2013-2015: Secretary of “Hack your Research” association</li>
                                <p>To promote Open Access of scientific publications</p>
                                <li>2013-2015: Secretary of “Hack your Research” association</li>
                                <p>Open Knowledge Conference, Geneva, Switzerland</p>
                                 <li>2013-2015: Secretary of “Hack your Research” association</li>
                                <li>Open Knowledge Conference, Geneva, Switzerland</li>
                                <li>2012-2015: Active member of “Cognivence” and « FRESCO » </li>
                                <p>French Federation of Cognitive Science students)</p>

                            </div>
                            <div class="tab-pane fade" id="tab2-item3">
                                <li>Peer-reviewing</li>
                                <p>2020: Molecular Autism<br>
                                    2019: Neuroimage (2 papers)</p>
                                <li>Computer skills</li>
                                <p>Programming  languages: R, Python, Matlab; Html, JavaScript, CSS (Web development).<br>
                                    Softwares: NIAK (fMRI), SPM12 (VBM, Neuromorphometric), Freesurfer and Civet (Cortical Thickness),  FSL (TBSS),  EEGlab (EEG)<br>
                                Github, Jupyter Notebook</p>
                                <li>Language</li>
                                <p>French (native), English (fluent)</p>
                                <li>Other experiences and skills</li>
                                <p>Job: Summers 2010, 2011 and 2012: Saleswoman in Paris, “Berthillon” (full time)<br>
                                    Sept 2010 to June 2012: Waitress in a restaurant in Paris (half-time)<br>
                                    Others: Driving license (car (2011), motorboat (2016), sailboat (2017)), Diving license (PADI, 2018), and Violin player.</p>
                                                                
                                <li class="page-header">Feindel Virtual Brain and Mind Seminar</li>
                                <iframe width="560" height="315" src="https://www.youtube.com/embed/P6450HqNKyo" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe> 
                            </div>
                        </div>
                    </div>
                </div>
            </div>

    <!--/#page-breadcrumb-->

    <section id="services">
        <div class="container">
            <div class="row">
                <div id="accordion-container">
                <h2 class="page-header"><img src="images/article.png">   Articles  </h2> 
                 
                <div class="panel-group" id="accordion">
                    <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
                    <div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapseOne">
                                    The general impact of haploinsufficiency on brain connectivity underlies the pleiotropic effect of neuropsychiatric CNVs.
                                </a>
                                 <div class='altmetric-embed' data-badge-type='4' data-doi="10.1101/2020.03.18.20038505"></div>
                            </h4>
                        </div>
                        <div id="collapseOne" class="panel-collapse collapse in">
                            <div class="panel-body">
                                <p>Moreau, C., Huguet, G., Urchs, S., Douard, E. A., Sharmarke, H., Orban, P., Labbe, A., Modenato, C., Martin-Brevet, S., Kuldeep, K., Martin, C.-O., Jizi, K., Younis, N., Tamer, P., Martineau, J.-L., Silva, A. I., Jonch, A. E., Lin, A., Simons VIP Foundation, ..., Thompson, P.M., Bellec, P., Jacquemont, S. (2020)</p>
                                <p> <a target="_blank" href="http://dx.doi.org/10.1101/2020.03.18.20038505">medRxiv 2020</a></p>
                                <p>Large effect-size mutations such as copy number variants (CNVs) have the potential to provide key insights into the underlying biological mechanisms linking deleterious genetic variants to brain architecture and neuropsychiatric disorders. To date, the effect of CNVs on functional brain connectivity remains mostly unstudied, and findings are derived from analyses conducted one mutation at a time. The lack of systematic cross-CNV comparisons hinders our understanding of any potential general mechanisms linking CNVs to effects on brain organization. We performed connectome-wide analyses using resting-state functional MRI data from 436 carriers of neuropsychiatric CNVs at the 1q21.1, 15q11.2, 16p11.2, 22q11.2 loci, and 4 neutral-effect CNVs, 66 carriers of scarcer neuropsychiatric CNVs, 756 individuals with idiopathic autism spectrum disorder (ASD), schizophrenia, attention deficit hyperactivity disorder, and 5,377 controls. Neuropsychiatric CNVs showed global shifts of mean connectivity. The effect size of CNVs on relative connectivity (adjusted for the mean) was correlated with the known level of neuropsychiatric risk conferred by CNVs. We reported architecture similarities between individuals with idiopathic psychiatric conditions and high-risk neuropsychiatric-CNVs, predominantly in the thalamus, the posterior cingulate cortex, and the anterior insula. We identified a linear relationship between connectivity and intolerance to haploinsufficiency measured for all genes encompassed by CNVs across 18 loci. This profile involved the thalamus, the basal ganglia, somatomotor and frontoparietal networks and was correlated with lower general intelligence and higher autism severity scores. An exploratory factor analysis confirmed the contribution of these regions to three latent components shared across CNVs and neuropsychiatric disorders. We posit that deleting genes intolerant to haploinsufficiency reorganize connectivity along general dimensions irrespective of where deletions occur in the genome. This haploinsufficiency brain signature opens new avenues to understand polygenicity in psychiatric conditions and the pleiotropic effect of CNVs on cognition and risk for neuropsychiatric disorders.</p>
                            </div>
                        </div>
                    </div>
                    <div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapseTwo">
                                    Neuropsychiatric mutations delineate functional brain connectivity dimensions contributing to autism and schizophrenia.
                                </a>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.1101/862615"></div>
                            </h4>
                        </div>
                        <div id="collapseTwo" class="panel-collapse collapse">
                            <div class="panel-body">
                                <p>Moreau, C., Urchs, S., Orban, P., Schramm, C., Dumas, G., Labbe, A., Huguet, G., Douard, E., Quirion, P.-O., Lin, A., Kushan, L., Grot, S., Luck, D., Mendrek, A., Potvin, S., Stip, E., Bourgeron, T., Evans, A. C., SimonsVIP Consortium, Bearden, CE., Bellec, P., Jacquemont, S.  </p>
                                <p> <a target="_blank" href="https://doi.org/10.1101/862615">bioRxiv 2019</a></p>

                                <p> 16p11.2 and 22q11.2 Copy Number Variants (CNVs) confer high risk for Autism Spectrum Disorder (ASD), schizophrenia (SZ), and Attention-Deficit-Hyperactivity-Disorder (ADHD), but their impact on functional connectivity (FC) networks remains unclear.We analyzed resting-state functional magnetic resonance imaging data from 101 CNV carriers, 755 individuals with idiopathic ASD, SZ, or ADHD and 1,072 controls. We used CNV FC-signatures to identify major dimensions contributing to complex idiopathic conditions. CNVs had large mirror effects on FC at the global and regional level, and their effect-sizes were twice as large as those of idiopathic conditions. Thalamus, somatomotor, and posterior insula regions played a critical role in dysconnectivity shared across deletions, duplications, idiopathic ASD, SZ but not ADHD. Individuals with higher similarity to deletion FC-signatures exhibited worse behavioral and cognitive symptoms. These seemingly distinct neuropsychiatric mutations showed similar gene co-expression patterns and converged on FC dimensions, that may represent mechanistic building blocks shared across idiopathic conditions.</p>
                            </div>
                        </div>
                    </div>
                    <div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapseThree">
                                    Quantifying the Effects of 16p11.2 Copy Number Variants on Brain Structure: A Multisite Genetic-First Study.
                                </a>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.1016/j.biopsych.2018.02.1176"></div>

                            </h4>
                        </div>
                       <div id="collapseThree" class="panel-collapse collapse">
                            <div class="panel-body">
                                <p>Martin-Brevet, S., Rodríguez-Herreros, B., Nielsen, J. A., Moreau, C., Modenato, C., Maillard, A. M., Pain, A., Richetin, S., Jønch, A. E., Qureshi, A. Y., Zürcher, N. R., Conus, P., 16p11.2 European Consortium, Simons Variation in Individuals Project (VIP) Consortium, Chung, W. K., Sherr, E. H., Spiro, J. E., Kherif, F., Beckmann, J. S., … Jacquemont, S. </p>
                                <p> <a target="_blank" href="https://doi.org/10.1016/j.biopsych.2018.02.1176">Biological Psychiatry 2018</a></p>

                                <p>16p11.2 breakpoint 4 to 5 copy number variants (CNVs) increase the risk for developing autism spectrum disorder, schizophrenia, and language and cognitive impairment. In this multisite study, we aimed to quantify the effect of 16p11.2 CNVs on brain structure. Using voxel- and surface-based brain morphometric methods, we analyzed structural magnetic resonance imaging collected at seven sites from 78 individuals with a deletion, 71 individuals with a duplication, and 212 individuals without a CNV. Beyond the 16p11.2-related mirror effect on global brain morphometry, we observe regional mirror differences in the insula (deletion > control > duplication). Other regions are preferentially affected by either the deletion or the duplication: the calcarine cortex and transverse temporal gyrus (deletion > control; Cohen’s d > 1), the superior and middle temporal gyri (deletion  > control; Cohen’s d inf −1), and the caudate and hippocampus (control > duplication; −0.5 > Cohen’s d > −1). Measures of cognition, language, and social responsiveness and the presence of psychiatric diagnoses do not influence these results. The global and regional effects on brain morphometry due to 16p11.2 CNVs generalize across site, computational method, age, and sex. Effect sizes on neuroimaging and cognitive traits are comparable. Findings partially overlap with results of meta-analyses performed across psychiatric disorders. However, the lack of correlation between morphometric and clinical measures suggests that CNV-associated brain changes contribute to clinical manifestations but require additional factors for the development of the disorder. These findings highlight the power of genetic risk factors as a complement to studying groups defined by behavioral criteria.</p>
                            </div>
                        </div>
                      </div>  
                    <div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapseFour">
                                    Estimating the effect size of the 15Q11.2 BP1-BP2 deletion and its contribution to neurodevelopmental symptoms: recommendations for practice.
                                </a>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.1136/jmedgenet-2018-105879"></div>

                            </h4>
                        </div>
                        <div id="collapseFour" class="panel-collapse collapse">
                            <div class="panel-body">
                                <p>Jønch, A. E., Douard, E., Moreau, C., Van Dijck, A., Passeggeri, M., Kooy, F., Puechberty, J., Campbell, C., Sanlaville, D., Lefroy, H., Richetin, S., Pain, A., Geneviève, D., Kini, U., Le Caignec, C., Lespinasse, J., Skytte, A.-B., Isidor, B., Zweier, C., ..., Ousager LB.,  Jacquemont, S., on behalf of 15q11.2 Working Group </p>
                                <p> <a target="_blank" href="https://doi.org/10.1136/jmedgenet-2018-105879">Journal of Medical Genetics 2019</a></p>

                                <p>The 15q11.2 deletion is frequently identified in the neurodevelopmental clinic. Case–control studies have associated the 15q11.2 deletion with neurodevelopmental disorders, and clinical case series have attempted to delineate a microdeletion syndrome with considerable phenotypic variability. The literature on this deletion is extensive and confusing, which is a challenge for genetic counselling. The aim of this study was to estimate the effect size of the 15q11.2 deletion and quantify its contribution to neurodevelopmental disorders. Methods We performed meta-analyses on new and previously published case–control studies and used statistical models trained in unselected populations with cognitive assessments. We used new (n=241) and previously published (n=150) data from a clinically referred group of deletion carriers. 15q11.2 duplications (new n=179 and previously published n=35) were used as a neutral control variant. The deletion decreases IQ by 4.3 points. The estimated ORs and respective frequencies in deletion carriers for intellectual disabilities, schizophrenia and epilepsy are 1.7 (3.4%), 1.5 (2%) and 3.1 (2.1%), respectively. There is no increased risk for heart malformations and autism. In the clinically referred group, the frequency and nature of symptoms in deletions are not different from those observed in carriers of the 15q11.2 duplication suggesting that most of the reported symptoms are due to ascertainment bias. We recommend that the deletion should be classified as ‘pathogenic of mild effect size’. Since it explains only a small proportion of the phenotypic variance in carriers, it is not worth discussing in the developmental clinic or in a prenatal setting.</p>
                            </div>
                        </div>
                      </div> 
                     <div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapseFive">
                                    MIST: A multi-resolution parcellation of functional brain networks.
                                </a>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.12688/mniopenres.12767.1"></div>

                            </h4>
                        </div>
                        <div id="collapseFive" class="panel-collapse collapse">
                            <div class="panel-body">
                                <p>Urchs, S., Armoza, J., Moreau, C., Benhajali, Y., St-Aubin, J., Orban, P., Bellec, P. </p>
                                <p> <a target="_blank" href="https://doi.org/10.12688/mniopenres.12767.2">MNI Open Research 2019</a></p>

                                <p>The functional architecture of the brain is organized across multiple levelsof spatial resolutions, from distributed networks to the localized areas theyare made of. A brain parcellation that defines functional nodes at multipleresolutions is required to investigate the functional connectome acrossthese scales. Here we present the Multiresolution Intrinsic SegmentationTemplate (MIST), a multi-resolution group level parcellation of the cortical,subcortical and cerebellar gray matter. The individual MIST parcellationsmatch other published group parcellations in internal homogeneity andreproducibility and perform very well in real-world application benchmarks.In addition, the MIST parcellations are fully annotated and provide ahierarchical decomposition of functional brain networks across nineresolutions (7 to 444 functional parcels). We hope that the MISTparcellation will accelerate research in brain connectivity acrossresolutions. Because visualizing multiresolution parcellations is challenging,we provide  to explore the MIST. The MIST isan interactive web interfacealso available through the popular toolbox.</p>
                            </div>
                        </div>    
                      </div>  
                    <div class="panel panel-default">
                        <div class="panel-heading">
                            <h4 class="panel-title">
                                <a data-toggle="collapse" data-parent="#accordion" href="#collapseSix">
                                    Association of Copy Number Variation of the 15q11.2 BP1-BP2 Region With Cortical and Subcortical Morphology and Cognition.
                                </a>
                                <div class='altmetric-embed' data-badge-type='4' data-doi="10.1001/jamapsychiatry.2019.3779"></div>

                            </h4>
                        </div>
                        <div id="collapseSix" class="panel-collapse collapse">
                            <div class="panel-body">
                                <p>van der Meer, D., Sønderby, I. E., Kaufmann, T., Walters, G. B., Abdellaoui, A., Ames, D., Amunts, K., Andersson, M., Armstrong, N. J., Bernard, M., Blackburn, N. B., Blangero, J., Boomsma, D. I., Brodaty, H., Brouwer, R. M., Bülow, R., Cahn, W., Calhoun, V. D., Caspers, S., ..., Moreau C., … Andreassen, O. A.</p>
                                <p> <a target="_blank" href="https://doi.org/10.1001/jamapsychiatry.2019.3779">JAMA Psychiatry 2019</a></p>

                                <p>Recurrent microdeletions and duplications in the genomic region 15q11.2 between breakpoints 1 (BP1) and 2 (BP2) are associated with neurodevelopmental disorders. These structural variants are present in 0.5% to 1.0% of the population, making 15q11.2 BP1-BP2 the site of the most prevalent known pathogenic copy number variation (CNV). It is unknown to what extent this CNV influences brain structure and affects cognitive abilities. To determine the association of the 15q11.2 BP1-BP2 deletion and duplication CNVs with cortical and subcortical brain morphology and cognitive task performance. In this genetic association study, T1-weighted brain magnetic resonance imaging were combined with genetic data from the ENIGMA-CNV consortium and the UK Biobank, with a replication cohort from Iceland. In total, 203 deletion carriers, 45 247 noncarriers, and 306 duplication carriers were included. Data were collected from August 2015 to April 2019, and data were analyzed from September 2018 to September 2019. The associations of the CNV with global and regional measures of surface area and cortical thickness as well as subcortical volumes were investigated, correcting for age, age2, sex, scanner, and intracranial volume. Additionally, measures of cognitive ability were analyzed in the full UK Biobank cohort. Of 45 756 included individuals, the mean (SD) age was 55.8 (18.3) years, and 23 754 (51.9%) were female. Compared with noncarriers, deletion carriers had a lower surface area (Cohen d = -0.41; SE, 0.08; P = 4.9 × 10-8), thicker cortex (Cohen d = 0.36; SE, 0.07; P = 1.3 × 10-7), and a smaller nucleus accumbens (Cohen d = -0.27; SE, 0.07; P = 7.3 × 10-5). There was also a significant negative dose response on cortical thickness (β = -0.24; SE, 0.05; P = 6.8 × 10-7). Regional cortical analyses showed a localization of the effects to the frontal, cingulate, and parietal lobes. Further, cognitive ability was lower for deletion carriers compared with noncarriers on 5 of 7 tasks. These findings, from the largest CNV neuroimaging study to date, provide evidence that 15q11.2 BP1-BP2 structural variation is associated with brain morphology and cognition, with deletion carriers being particularly affected. The pattern of results fits with known molecular functions of genes in the 15q11.2 BP1-BP2 region and suggests involvement of these genes in neuronal plasticity. These neurobiological effects likely contribute to the association of this CNV with neurodevelopmental disorders.</p>
                            </div>
                        </div>    
                      </div>
        
                    </div>
                </div><!--/#accordion-->
            </div><!--/#accordion-container-->
<!--/#table-container-->
            </div>
        </div>
</section>
        
    
                
    <section id="services">
       
        <div class="container">
            <div class="row">
             <h2 class="page-header"><img src="images/brain.png">   Media</h2>
                                
                <div class="col-sm-4 text-center padding wow fadeIn" >
                    <div class="single-service">
                        <div class="wow scaleIn" >
                        </div>
                       <blockquote class="twitter-tweet"><p lang="en" dir="ltr">Neuropsychiatric <a href="https://twitter.com/hashtag/CNVs?src=hash&amp;ref_src=twsrc%5Etfw">#CNVs</a> across the genome reorganize brain <a href="https://twitter.com/hashtag/connectivity?src=hash&amp;ref_src=twsrc%5Etfw">#connectivity</a> architecture along<br>dominant patterns contributing to complex idiopathic <a href="https://twitter.com/hashtag/psychiatric?src=hash&amp;ref_src=twsrc%5Etfw">#psychiatric</a> conditions. <a href="https://twitter.com/hashtag/pleiotropy?src=hash&amp;ref_src=twsrc%5Etfw">#pleiotropy</a><a href="https://twitter.com/JacquemontSeb?ref_src=twsrc%5Etfw">@JacquemontSeb</a> <a href="https://twitter.com/pierre_bellec?ref_src=twsrc%5Etfw">@pierre_bellec</a> labs<a href="https://twitter.com/s_urchs?ref_src=twsrc%5Etfw">@s_urchs</a> <a href="https://twitter.com/PTenigma?ref_src=twsrc%5Etfw">@PTenigma</a> <a href="https://twitter.com/Winkytheelf?ref_src=twsrc%5Etfw">@Winkytheelf</a> (...)<a href="https://t.co/2uopJY9zZZ">https://t.co/2uopJY9zZZ</a> <a href="https://t.co/lqxhOHdSOW">pic.twitter.com/lqxhOHdSOW</a></p>&mdash; Clara Moreau (@ClaraMoreau9) <a href="https://twitter.com/ClaraMoreau9/status/1242096202881159168?ref_src=twsrc%5Etfw">March 23, 2020</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
                    </div>
                </div>
                <div class="col-sm-4 text-center padding wow fadeIn">
                    <div class="single-service">
                        <div class="wow scaleIn">
                        </div>
                        <blockquote class="twitter-tweet"><p lang="en" dir="ltr">Does your GENETIC CODE affect your BRAIN connectivity? new discoveries shed light on autism, ADHD, schizophrenia; GREAT new lecture by Clara Moreau <br> <a href="https://t.co/JBN3qX1yO2">https://t.co/JBN3qX1yO2</a> - with <a href="https://twitter.com/JacquemontSeb?ref_src=twsrc%5Etfw">@JacquemontSeb</a> <a href="https://twitter.com/pierre_bellec?ref_src=twsrc%5Etfw">@pierre_bellec</a> <a href="https://t.co/QeDXCtG2am">https://t.co/QeDXCtG2am</a> <a href="https://t.co/tt4aA9icxT">pic.twitter.com/tt4aA9icxT</a></p>&mdash; Paul Thompson (@PTenigma) <a href="https://twitter.com/PTenigma/status/1256041670291755008?ref_src=twsrc%5Etfw">May 1, 2020</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
                    </div>
                </div>
                 <div class="col-sm-4 text-center padding wow fadeIn" >
                   <div class="single-service">
                        <div class="wow scaleIn">
                        </div>
                  <blockquote class="twitter-tweet"><p lang="en" dir="ltr">huge congrats to <a href="https://twitter.com/ClaraMoreau9?ref_src=twsrc%5Etfw">@ClaraMoreau9</a> on her first talk at a major workshop <a href="https://twitter.com/AutismINSAR?ref_src=twsrc%5Etfw">@AutismINSAR</a> !! Many more will follow without a doubt, so honoured and humbled I get to work with such an amazing student! <a href="https://t.co/9Jl5E3LhjA">pic.twitter.com/9Jl5E3LhjA</a></p>&mdash; Pierre Bellec (@pierre_bellec) <a href="https://twitter.com/pierre_bellec/status/1124774214710562817?ref_src=twsrc%5Etfw">May 4, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
             </div>
            </div>
        </div>
    </section>
    
      </section>
    <!--/#clients-->

 
    
    <footer id="footer">
        <div class="container">
            <div class="row">
            <div class="col-sm-12">
                    <div class="copyright-text text-center">
                        <h2>Contacts</h2>
                        <address>
                        E-mail: <a href="mailto:someone@example.com">clara.moreau@umontreal.ca</a> <br> 
                        Phone: +1 (438) 763 6198 <br> 
                        </address>

                        <h2>Address</h2>
                        <address>
                        Montreal, QC <br> 
                        H2e1z3, <br> 
                        Canada <br> 
                        </address>
                        <br><br>
                        <p>&copy; 2020.</p>
                        <p>Adapted from <a target="_blank" href="http://www.themeum.com">Themeum</a></p>
                    </div>
            </div>
            </div>
        </div>
    </footer>
    <!--/#footer-->

    <script type="text/javascript" src="js/jquery.js"></script>
    <script type="text/javascript" src="js/bootstrap.min.js"></script>
    <script type="text/javascript" src="js/lightbox.min.js"></script>
    <script type="text/javascript" src="js/wow.min.js"></script>
    <script type="text/javascript" src="js/main.js"></script>   
</body>
</html>
